---
title: "Artificial intelligence in digital pathology — time for a reality check - Nature Reviews Clinical Oncology"
source: "https://www.nature.com/articles/s41571-025-00991-6"
author:
  - "[[Arpit Aggarwal]]"
  - "[[Satvika Bharadwaj]]"
  - "[[Germán Corredor]]"
  - "[[Tilak Pathak]]"
  - "[[Sunil Badve]]"
  - "[[Anant Madabhushi]]"
published: 2025-02-11
created: 2025-02-11
description: "The past decade has seen the introduction of artificial intelligence (AI)-based approaches aimed at optimizing several workflows across many medical specialties. In clinical oncology, the most promising applications include those involving image analysis, such as digital pathology. In this Perspective, we provide a comprehensive examination of the developments in AI in digital pathology between 2019 and 2024. We evaluate the current landscape from the lens of technological innovations, regulatory trends, deployment and implementation, reimbursement and commercial implications. We assess the technological advances that have driven improvements in AI, enabling more robust and scalable solutions for digital pathology. We also examine regulatory developments, in particular those affecting in-house devices and laboratory-developed tests, which are shaping the landscape of AI-based tools in digital pathology. Finally, we discuss the role of reimbursement frameworks and commercial investment in the clinical adoption of AI-based technologies. In this Perspective, we highlight both the progress and challenges in AI-driven digital pathology over the past 5 years, outlining the path forward for its adoption into routine practice in clinical oncology. The authors of this Perspective evaluate the developments in the use of artificial intelligence (AI) in digital pathology for oncology applications between 2019 and 2024, addressing technological innovations, regulatory trends, implementation and financial implications. Importantly, they explore the current landscape of clinical deployment, highlighting future opportunities for the integration of AI into clinical oncology routine practice."
tags:
  - "clippings"
---
## Introduction

The integration of artificial intelligence (AI) into routine health-care applications has the potential to drive substantial advances across various medical fields, improving diagnostic precision, workflow efficiency and prediction of patient prognosis<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Rajpurkar, P., Chen, E., Banerjee, O. &amp; Topol, E. J. AI in health and medicine. Nat. Med. 28, 31–38 (2022)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR1" id="ref-link-section-d5807050e380">1</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Topol, E. J. High-performance medicine: the convergence of human and artificial intelligence. Nat. Med. 25, 44–56 (2019)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR2" id="ref-link-section-d5807050e383">2</a></sup>. Pathology constitutes the diagnostic foundation of oncology, enabling precise cancer identification and characterization, which are essential for developing effective treatment plans and advancing cancer research<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Aeffner, F. et al. Introduction to digital image analysis in whole-slide imaging: a white paper from the Digital Pathology Association. J. Pathol. Inform. 10, 9 (2019)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR3" id="ref-link-section-d5807050e387">3</a></sup>. Traditionally reliant on the visual examination of tissue samples under a microscope, pathology has dramatically evolved with the advent of digitized images<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Aeffner, F. et al. Introduction to digital image analysis in whole-slide imaging: a white paper from the Digital Pathology Association. J. Pathol. Inform. 10, 9 (2019)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR3" id="ref-link-section-d5807050e391">3</a></sup>. This shift has enabled further innovations, including the integration of AI<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="van der Laak, J., Litjens, G. &amp; Ciompi, F. Deep learning in histopathology: the path to the clinic. Nat. Med. 27, 775–784 (2021)." href="https://www.nature.com/articles/#ref-CR4" id="ref-link-section-d5807050e395">4</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Shafi, S. &amp; Parwani, A. V. Artificial intelligence in diagnostic pathology. Diagn. Pathol. 18, 109 (2023)." href="https://www.nature.com/articles/#ref-CR5" id="ref-link-section-d5807050e395_1">5</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Baxi, V., Edwards, R., Montalto, M. &amp; Saha, S. Digital pathology and artificial intelligence in translational medicine and clinical practice. Mod. Pathol. 35, 23–32 (2022)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR6" id="ref-link-section-d5807050e398">6</a></sup>. As these technologies advance, we must assess the extent of AI adoption in clinical pathology — particularly in oncology-related applications — and identify the barriers impeding broader implementation.

In 2019, in collaboration with other experts, we evaluated a range of AI-based approaches with applications in digital pathology for oncology, and predicted major developments across key areas of the field<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Bera, K., Schalper, K. A., Rimm, D. L., Velcheti, V. &amp; Madabhushi, A. Artificial intelligence in digital pathology – new tools for diagnosis and precision oncology. Nat. Rev. Clin. Oncol. 16, 703–715 (2019)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR7" id="ref-link-section-d5807050e405">7</a></sup>. We anticipated advances in technology, particularly improvements in the efficiency and accuracy of diagnostics and prognostics based on image analysis. We also foresaw regulatory changes and, focusing on the FDA, predicted an increase in the number of approvals and the introduction of reimbursement codes for AI-related devices in the USA, which would increase the clinical acceptance of digital pathology with similar advances in other regions. Furthermore, we emphasized the importance of integrating AI into clinical workflows through trials, and predicted that commercial investment by key companies would drive further innovation.

In this Perspective, we revisit the projections we made 5 years ago for the field of digital pathology in oncology, assessing whether they have been realized and identifying the challenges that have hindered their fulfilment, with an emphasis on AI advances and adoption beyond the hurdles in the uptake of digital pathology itself. We begin by analysing technological advances that address key technical challenges, such as data annotation, interpretability and computational complexity. We then examine the role of regulatory bodies, comparing approvals of applications for digital pathology and other medical specialties. We discuss changes in reimbursement frameworks, including the introduction of new billing codes and developments for in vitro diagnostic tools — in particular, [in-house devices](https://www.nature.com/articles/s41571-025-00991-6#Glos1) and [laboratory-developed tests](https://www.nature.com/articles/s41571-025-00991-6#Glos2) (LDTs). Finally, we explore the current landscape of clinical deployment, highlighting ongoing trials investigating the integration of AI into routine practice in clinical oncology.

## Advances in AI model development

The landscape of AI in digital pathology has evolved substantially in the past 5 years, shifting from an initial reliance on [convolutional neural networks](https://www.nature.com/articles/s41571-025-00991-6#Glos3) (CNNs)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="O’Shea, K. &amp; Nash, R. An introduction to convolutional neural networks. Preprint at arXiv 
                https://doi.org/10.48550/arXiv.1511.08458
                
               (2015)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR8" id="ref-link-section-d5807050e429">8</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Coudray, N. et al. Classification and mutation prediction from non–small cell lung cancer histopathology images using deep learning. Nat. Med. 24, 1559–1567 (2018)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR9" id="ref-link-section-d5807050e432">9</a></sup> to the adoption of more advanced technologies, such as [transformers](https://www.nature.com/articles/s41571-025-00991-6#Glos4)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Dosovitskiy, A. et al. An image is worth 16x16 words: transformers for image recognition at scale. Preprint at arXiv 
                https://doi.org/10.48550/arXiv.2010.11929
                
               (2020)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR10" id="ref-link-section-d5807050e438">10</a></sup> and subsequently [foundation models](https://www.nature.com/articles/s41571-025-00991-6#Glos5)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Chen, R. J. et al. Towards a general-purpose foundation model for computational pathology. Nat. Med. 30, 850–862 (2024)." href="https://www.nature.com/articles/#ref-CR11" id="ref-link-section-d5807050e444">11</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Xu, H. et al. A whole-slide foundation model for digital pathology from real-world data. Nature 630, 181–188 (2024)." href="https://www.nature.com/articles/#ref-CR12" id="ref-link-section-d5807050e444_1">12</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Vorontsov, E. A foundation model for clinical-grade computational pathology and rare cancers detection. Nat. Med. 30, 2924–2935 (2024)." href="https://www.nature.com/articles/#ref-CR13" id="ref-link-section-d5807050e444_2">13</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Dippel, J. et al. RudolfV: a foundation model by pathologists for pathologists. Preprint at arXiv 
                https://doi.org/10.48550/arXiv.2401.04079
                
               (2024)." href="https://www.nature.com/articles/#ref-CR14" id="ref-link-section-d5807050e444_3">14</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Wang, X. et al. A pathology foundation model for cancer diagnosis and prognosis prediction. Nature 634, 970–978 (2024)." href="https://www.nature.com/articles/#ref-CR15" id="ref-link-section-d5807050e444_4">15</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Ma, J. et al. Towards a generalizable pathology foundation model via unified knowledge distillation. Preprint at arXiv 
                https://doi.org/10.48550/arXiv.2407.18449
                
               (2024)." href="https://www.nature.com/articles/#ref-CR16" id="ref-link-section-d5807050e444_5">16</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Yang, Z. et al. A foundation model for generalizable cancer diagnosis and survival prediction from histopathological images. Preprint at bioRxiv 
                https://doi.org/10.1101/2024.05.16.594499
                
               (2024)." href="https://www.nature.com/articles/#ref-CR17" id="ref-link-section-d5807050e444_6">17</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Hua, S., Yan, F., Shen, T., Ma, L. &amp; Zhang, X. PathoDuet: foundation models for pathological slide analysis of H&amp;E and IHC stains. Med. Image Anal. 97, 103289 (2024)." href="https://www.nature.com/articles/#ref-CR18" id="ref-link-section-d5807050e444_7">18</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Nechaev, D., Pchelnikov, A. &amp; Ivanova, E. Hibou: a family of foundational vision transformers for pathology. Preprint at arXiv 
                https://doi.org/10.48550/arXiv.2406.05074
                
               (2024)." href="https://www.nature.com/articles/#ref-CR19" id="ref-link-section-d5807050e444_8">19</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Lu, M. Y. et al. A visual-language foundation model for computational pathology. Nat. Med. 30, 863–874 (2024)." href="https://www.nature.com/articles/#ref-CR20" id="ref-link-section-d5807050e444_9">20</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Zimmermann, E. et al. Virchow2: scaling self-supervised mixed magnification models in pathology. Preprint at arXiv 
                https://doi.org/10.48550/arXiv.2408.00738
                
               (2024)." href="https://www.nature.com/articles/#ref-CR21" id="ref-link-section-d5807050e444_10">21</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Juyal, D. et al. PLUTO: pathology-universal transformer. Preprint at arXiv 
                https://doi.org/10.48550/arXiv.2405.07905
                
               (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR22" id="ref-link-section-d5807050e447">22</a></sup>, to perform various tasks including predicting patient prognosis and response to treatment<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Chen, R. J. et al. Towards a general-purpose foundation model for computational pathology. Nat. Med. 30, 850–862 (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR11" id="ref-link-section-d5807050e452">11</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Vorontsov, E. A foundation model for clinical-grade computational pathology and rare cancers detection. Nat. Med. 30, 2924–2935 (2024)." href="https://www.nature.com/articles/#ref-CR13" id="ref-link-section-d5807050e455">13</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Dippel, J. et al. RudolfV: a foundation model by pathologists for pathologists. Preprint at arXiv 
                https://doi.org/10.48550/arXiv.2401.04079
                
               (2024)." href="https://www.nature.com/articles/#ref-CR14" id="ref-link-section-d5807050e455_1">14</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Wang, X. et al. A pathology foundation model for cancer diagnosis and prognosis prediction. Nature 634, 970–978 (2024)." href="https://www.nature.com/articles/#ref-CR15" id="ref-link-section-d5807050e455_2">15</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Ma, J. et al. Towards a generalizable pathology foundation model via unified knowledge distillation. Preprint at arXiv 
                https://doi.org/10.48550/arXiv.2407.18449
                
               (2024)." href="https://www.nature.com/articles/#ref-CR16" id="ref-link-section-d5807050e455_3">16</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Yang, Z. et al. A foundation model for generalizable cancer diagnosis and survival prediction from histopathological images. Preprint at bioRxiv 
                https://doi.org/10.1101/2024.05.16.594499
                
               (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR17" id="ref-link-section-d5807050e458">17</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Li, H. et al. A prognostic and predictive computational pathology immune signature for ductal carcinoma in situ: retrospective results from a cohort within the UK/ANZ DCIS trial. Lancet Digital Health 6, e562–e569 (2024)." href="https://www.nature.com/articles/#ref-CR23" id="ref-link-section-d5807050e461">23</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Lu, M. Y. et al. AI-based pathology predicts origins for cancers of unknown primary. Nature 594, 106–110 (2021)." href="https://www.nature.com/articles/#ref-CR24" id="ref-link-section-d5807050e461_1">24</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Bulten, W. et al. Artificial intelligence for diagnosis and Gleason grading of prostate cancer: the PANDA challenge. Nat. Med. 28, 154–163 (2022)." href="https://www.nature.com/articles/#ref-CR25" id="ref-link-section-d5807050e461_2">25</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Bulten, W. et al. Automated deep-learning system for Gleason grading of prostate cancer using biopsies: a diagnostic study. Lancet Oncol. 21, 233–241 (2020)." href="https://www.nature.com/articles/#ref-CR26" id="ref-link-section-d5807050e461_3">26</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Aggarwal, A. et al. Computational pathology identifies immune-mediated collagen disruption to predict clinical outcomes in gynecologic malignancies. Commun. Med. 4, 2 (2024)." href="https://www.nature.com/articles/#ref-CR27" id="ref-link-section-d5807050e461_4">27</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Li, H. et al. Collagen fiber orientation disorder from H&amp;E images is prognostic for early stage breast cancer: clinical trial validation. NPJ Breast Cancer 7, 104 (2021)." href="https://www.nature.com/articles/#ref-CR28" id="ref-link-section-d5807050e461_5">28</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Bilal, M. et al. Development and validation of a weakly supervised deep learning framework to predict the status of molecular pathways and key mutations in colorectal cancer from routine histology images: a retrospective study. Lancet Digital Health 3, e763–e772 (2021)." href="https://www.nature.com/articles/#ref-CR29" id="ref-link-section-d5807050e461_6">29</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Lugli, A., Zlobec, I., Berger, M. D., Kirsch, R. &amp; Nagtegaal, I. D. Tumour budding in solid cancers. Nat. Rev. Clin. Oncol. 18, 101–115 (2021)." href="https://www.nature.com/articles/#ref-CR30" id="ref-link-section-d5807050e461_7">30</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Neppl, C., Zlobec, I., Schmid, R. A. &amp; Berezowska, S. Validation of the International Tumor Budding Consensus Conference (ITBCC) 2016 recommendation in squamous cell carcinoma of the lung – a single-center analysis of 354 cases. Mod. Pathol. 33, 802–811 (2020)." href="https://www.nature.com/articles/#ref-CR31" id="ref-link-section-d5807050e461_8">31</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Dawson, H. et al. Tumour budding/T cell infiltrates in colorectal cancer: proposal of a novel combined score. Histopathology 76, 572–580 (2020)." href="https://www.nature.com/articles/#ref-CR32" id="ref-link-section-d5807050e461_9">32</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Xie, W. et al. Prostate cancer risk stratification via nondestructive 3D pathology with deep learning-assisted gland analysis. Cancer Res. 82, 334–345 (2022)." href="https://www.nature.com/articles/#ref-CR33" id="ref-link-section-d5807050e461_10">33</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Lu, C. et al. A prognostic model for overall survival of patients with early-stage non-small cell lung cancer: a multicentre, retrospective study. Lancet Digital Health 2, e594–e606 (2020)." href="https://www.nature.com/articles/#ref-CR34" id="ref-link-section-d5807050e461_11">34</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Lu, C. et al. Feature-driven local cell graph (FLocK): new computational pathology-based descriptors for prognosis of lung cancer and HPV status of oropharyngeal cancers. Med. Image Anal. 68, 101903 (2021)." href="https://www.nature.com/articles/#ref-CR35" id="ref-link-section-d5807050e461_12">35</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Fashi, P. A., Hemati, S., Babaie, M., Gonzalez, R. &amp; Tizhoosh, H. R. A self-supervised contrastive learning approach for whole slide image representation in digital pathology. J. Pathol. Inf. 13, 100133 (2022)." href="https://www.nature.com/articles/#ref-CR36" id="ref-link-section-d5807050e461_13">36</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Tsai, P.-C. et al. Histopathology images predict multi-omics aberrations and prognoses in colorectal cancer patients. Nat. Commun. 14, 2102 (2023)." href="https://www.nature.com/articles/#ref-CR37" id="ref-link-section-d5807050e461_14">37</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Marostica, E. et al. Development of a histopathology informatics pipeline for classification and prediction of clinical outcomes in subtypes of renal cell carcinoma. Clin. Cancer Res. 27, 2868–2878 (2021)." href="https://www.nature.com/articles/#ref-CR38" id="ref-link-section-d5807050e461_15">38</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Campanella, G., Vanderbilt, C. &amp; Fuchs, T. Computational pathology at health system scale – self-supervised foundation models from three billion images. Preprint at arXiv 
                https://doi.org/10.48550/arXiv.2310.07033
                
               (2024)." href="https://www.nature.com/articles/#ref-CR39" id="ref-link-section-d5807050e461_16">39</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Da Silva, L. M. et al. Independent real‐world application of a clinical‐grade automated prostate cancer detection system. J. Pathol. 254, 147–158 (2021)." href="https://www.nature.com/articles/#ref-CR40" id="ref-link-section-d5807050e461_17">40</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Corredor, G. et al. Spatial architecture and arrangement of tumor-infiltrating lymphocytes for predicting likelihood of recurrence in early-stage non-small cell lung cancer. Clin. Cancer Res. 25, 1526–1534 (2019)." href="https://www.nature.com/articles/#ref-CR41" id="ref-link-section-d5807050e461_18">41</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Koyuncu, C. F. et al. Computerized tumor multinucleation index (MuNI) is prognostic in p16+ oropharyngeal carcinoma. J. Clin. Invest. 131, e145488 (2021)." href="https://www.nature.com/articles/#ref-CR42" id="ref-link-section-d5807050e461_19">42</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Balkenhol, M. C. A. et al. Optimized tumour infiltrating lymphocyte assessment for triple negative breast cancer prognostics. Breast 56, 78–87 (2021)." href="https://www.nature.com/articles/#ref-CR43" id="ref-link-section-d5807050e461_20">43</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Pinckaers, H. et al. Predicting biochemical recurrence of prostate cancer with artificial intelligence. Commun. Med. 2, 64 (2022)." href="https://www.nature.com/articles/#ref-CR44" id="ref-link-section-d5807050e461_21">44</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Vendittelli, P. et al. Automatic quantification of tumor–stroma ratio as a prognostic marker for pancreatic cancer. PLoS ONE 19, e0301969 (2024)." href="https://www.nature.com/articles/#ref-CR45" id="ref-link-section-d5807050e461_22">45</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Aggarwal, A. et al. Image analysis uncovers associations between immune landscape, collagen structure, and neoadjuvant chemotherapy in high-grade serous ovarian carcinomas. Heliyon 10, e33618 (2024)." href="https://www.nature.com/articles/#ref-CR46" id="ref-link-section-d5807050e461_23">46</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Li, H. et al. Quantitative nuclear histomorphometric features are predictive of Oncotype DX risk categories in ductal carcinoma in situ: preliminary findings. Breast Cancer Res. 21, 114 (2019)." href="https://www.nature.com/articles/#ref-CR47" id="ref-link-section-d5807050e461_24">47</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Geaney, A. et al. Translation of tissue-based artificial intelligence into clinical practice: from discovery to adoption. Oncogene 42, 3545–3555 (2023)." href="https://www.nature.com/articles/#ref-CR48" id="ref-link-section-d5807050e461_25">48</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Balestriero, R. et al. A cookbook of self-supervised learning. Preprint at arXiv 
                https://doi.org/10.48550/arXiv.2304.12210
                
               (2023)." href="https://www.nature.com/articles/#ref-CR49" id="ref-link-section-d5807050e461_26">49</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Unger, M. &amp; Kather, J. N. A systematic analysis of deep learning in genomics and histopathology for precision oncology. BMC Med. Genomics 17, 48 (2024)." href="https://www.nature.com/articles/#ref-CR50" id="ref-link-section-d5807050e461_27">50</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Monabbati, S. et al. Machine vision-detected peritumoral lymphocytic aggregates are associated with disease-free survival in papillary thyroid carcinoma patients. Lab. Invest. 104, 102168 (2024)." href="https://www.nature.com/articles/#ref-CR51" id="ref-link-section-d5807050e461_28">51</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Monabbati, S. et al. A novel computational pathology approach for identifying gene signatures prognostic of disease-free survival for papillary thyroid carcinomas. Eur. J. Cancer 212, 114326 (2024)." href="https://www.nature.com/articles/#ref-CR52" id="ref-link-section-d5807050e461_29">52</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Wang, X. et al. Spatial interplay patterns of cancer nuclei and tumor-infiltrating lymphocytes (TILs) predict clinical benefit for immune checkpoint inhibitors. Sci. Adv. 8, eabn3966 (2022)." href="https://www.nature.com/articles/#ref-CR53" id="ref-link-section-d5807050e461_30">53</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Arabyarmohammadi, S. et al. in Medical Image Computing and Computer Assisted Intervention – MICCAI 2023 Part VI (eds Greenspan, H. et al.) 797–807 (Springer, 2023)." href="https://www.nature.com/articles/#ref-CR54" id="ref-link-section-d5807050e461_31">54</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Monabbati, S., Khalighi, S., Fu, P., Asa, S. &amp; Madabhushi, A. in Medical Imaging 2023: Digital and Computational Pathology Vol. 12471 (eds Tomaszewski, J. E. &amp; Ward, A. D.) 284–293 (SPIE, 2023)." href="https://www.nature.com/articles/#ref-CR55" id="ref-link-section-d5807050e461_32">55</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Nag, R. et al. Association of artificial intelligence-derived collagen disorder architecture (CoDA) features with survival outcome and objective response to immune checkpoint inhibitors in patients with head and neck squamous cell carcinoma [abstract]. J. Clin. Oncol. 42 (Suppl. 16), 6061 (2024)." href="https://www.nature.com/articles/#ref-CR56" id="ref-link-section-d5807050e461_33">56</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 57" title="Monabbati, S. et al. Automated analysis of computerized morphological features of cell clusters associated with malignancy on bile duct brushing whole slide images. Cancer Med. 12, 6365–6378 (2023)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR57" id="ref-link-section-d5807050e464">57</a></sup>. Initially, researchers predominantly used fully [supervised learning models](https://www.nature.com/articles/s41571-025-00991-6#Glos6) that required large annotated datasets for training<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Coudray, N. et al. Classification and mutation prediction from non–small cell lung cancer histopathology images using deep learning. Nat. Med. 24, 1559–1567 (2018)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR9" id="ref-link-section-d5807050e471">9</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Bulten, W. et al. Artificial intelligence for diagnosis and Gleason grading of prostate cancer: the PANDA challenge. Nat. Med. 28, 154–163 (2022)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR25" id="ref-link-section-d5807050e474">25</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Da Silva, L. M. et al. Independent real‐world application of a clinical‐grade automated prostate cancer detection system. J. Pathol. 254, 147–158 (2021)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR40" id="ref-link-section-d5807050e477">40</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 48" title="Geaney, A. et al. Translation of tissue-based artificial intelligence into clinical practice: from discovery to adoption. Oncogene 42, 3545–3555 (2023)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR48" id="ref-link-section-d5807050e480">48</a></sup>, including CNNs that were widely applied to tasks such as image classification, segmentation and detection<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Coudray, N. et al. Classification and mutation prediction from non–small cell lung cancer histopathology images using deep learning. Nat. Med. 24, 1559–1567 (2018)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR9" id="ref-link-section-d5807050e484">9</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 48" title="Geaney, A. et al. Translation of tissue-based artificial intelligence into clinical practice: from discovery to adoption. Oncogene 42, 3545–3555 (2023)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR48" id="ref-link-section-d5807050e487">48</a></sup>. However, a notable shift has occurred initially towards weakly supervised<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 58" title="Lu, M. Y. et al. Data-efficient and weakly supervised computational pathology on whole-slide images. Nat. Biomed. Eng. 5, 555–570 (2021)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR58" id="ref-link-section-d5807050e491">58</a></sup> and subsequently [self-supervised learning models](https://www.nature.com/articles/s41571-025-00991-6#Glos7)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 49" title="Balestriero, R. et al. A cookbook of self-supervised learning. Preprint at arXiv 
                https://doi.org/10.48550/arXiv.2304.12210
                
               (2023)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR49" id="ref-link-section-d5807050e498">49</a></sup>, which reduce the need for extensive manual annotation by enabling models to learn from large volumes of unlabelled data<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Campanella, G., Vanderbilt, C. &amp; Fuchs, T. Computational pathology at health system scale – self-supervised foundation models from three billion images. Preprint at arXiv 
                https://doi.org/10.48550/arXiv.2310.07033
                
               (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR39" id="ref-link-section-d5807050e502">39</a></sup>.

Foundation models constitute a substantial advance in computational pathology<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Chen, R. J. et al. Towards a general-purpose foundation model for computational pathology. Nat. Med. 30, 850–862 (2024)." href="https://www.nature.com/articles/#ref-CR11" id="ref-link-section-d5807050e509">11</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Xu, H. et al. A whole-slide foundation model for digital pathology from real-world data. Nature 630, 181–188 (2024)." href="https://www.nature.com/articles/#ref-CR12" id="ref-link-section-d5807050e509_1">12</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Vorontsov, E. A foundation model for clinical-grade computational pathology and rare cancers detection. Nat. Med. 30, 2924–2935 (2024)." href="https://www.nature.com/articles/#ref-CR13" id="ref-link-section-d5807050e509_2">13</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Dippel, J. et al. RudolfV: a foundation model by pathologists for pathologists. Preprint at arXiv 
                https://doi.org/10.48550/arXiv.2401.04079
                
               (2024)." href="https://www.nature.com/articles/#ref-CR14" id="ref-link-section-d5807050e509_3">14</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Wang, X. et al. A pathology foundation model for cancer diagnosis and prognosis prediction. Nature 634, 970–978 (2024)." href="https://www.nature.com/articles/#ref-CR15" id="ref-link-section-d5807050e509_4">15</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Ma, J. et al. Towards a generalizable pathology foundation model via unified knowledge distillation. Preprint at arXiv 
                https://doi.org/10.48550/arXiv.2407.18449
                
               (2024)." href="https://www.nature.com/articles/#ref-CR16" id="ref-link-section-d5807050e509_5">16</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Yang, Z. et al. A foundation model for generalizable cancer diagnosis and survival prediction from histopathological images. Preprint at bioRxiv 
                https://doi.org/10.1101/2024.05.16.594499
                
               (2024)." href="https://www.nature.com/articles/#ref-CR17" id="ref-link-section-d5807050e509_6">17</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Hua, S., Yan, F., Shen, T., Ma, L. &amp; Zhang, X. PathoDuet: foundation models for pathological slide analysis of H&amp;E and IHC stains. Med. Image Anal. 97, 103289 (2024)." href="https://www.nature.com/articles/#ref-CR18" id="ref-link-section-d5807050e509_7">18</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Nechaev, D., Pchelnikov, A. &amp; Ivanova, E. Hibou: a family of foundational vision transformers for pathology. Preprint at arXiv 
                https://doi.org/10.48550/arXiv.2406.05074
                
               (2024)." href="https://www.nature.com/articles/#ref-CR19" id="ref-link-section-d5807050e509_8">19</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Lu, M. Y. et al. A visual-language foundation model for computational pathology. Nat. Med. 30, 863–874 (2024)." href="https://www.nature.com/articles/#ref-CR20" id="ref-link-section-d5807050e509_9">20</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Zimmermann, E. et al. Virchow2: scaling self-supervised mixed magnification models in pathology. Preprint at arXiv 
                https://doi.org/10.48550/arXiv.2408.00738
                
               (2024)." href="https://www.nature.com/articles/#ref-CR21" id="ref-link-section-d5807050e509_10">21</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Juyal, D. et al. PLUTO: pathology-universal transformer. Preprint at arXiv 
                https://doi.org/10.48550/arXiv.2405.07905
                
               (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR22" id="ref-link-section-d5807050e512">22</a></sup>. A common training strategy for these models is self-supervised learning methods, for example, DINOv2 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 59" title="Oquab, M. et al. DINOv2: learning robust visual features without supervision. Preprint at arXiv 
                https://doi.org/10.48550/arXiv.2304.07193
                
               (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR59" id="ref-link-section-d5807050e516">59</a></sup>), which enables robust feature learning without requiring annotated data. Once trained, foundation models can be fine-tuned to perform specific pathology tasks, demonstrating promising performance in tasks such as image classification, segmentation and other applications<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Chen, R. J. et al. Towards a general-purpose foundation model for computational pathology. Nat. Med. 30, 850–862 (2024)." href="https://www.nature.com/articles/#ref-CR11" id="ref-link-section-d5807050e520">11</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Xu, H. et al. A whole-slide foundation model for digital pathology from real-world data. Nature 630, 181–188 (2024)." href="https://www.nature.com/articles/#ref-CR12" id="ref-link-section-d5807050e520_1">12</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Vorontsov, E. A foundation model for clinical-grade computational pathology and rare cancers detection. Nat. Med. 30, 2924–2935 (2024)." href="https://www.nature.com/articles/#ref-CR13" id="ref-link-section-d5807050e520_2">13</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Dippel, J. et al. RudolfV: a foundation model by pathologists for pathologists. Preprint at arXiv 
                https://doi.org/10.48550/arXiv.2401.04079
                
               (2024)." href="https://www.nature.com/articles/#ref-CR14" id="ref-link-section-d5807050e520_3">14</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Wang, X. et al. A pathology foundation model for cancer diagnosis and prognosis prediction. Nature 634, 970–978 (2024)." href="https://www.nature.com/articles/#ref-CR15" id="ref-link-section-d5807050e520_4">15</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Ma, J. et al. Towards a generalizable pathology foundation model via unified knowledge distillation. Preprint at arXiv 
                https://doi.org/10.48550/arXiv.2407.18449
                
               (2024)." href="https://www.nature.com/articles/#ref-CR16" id="ref-link-section-d5807050e520_5">16</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Yang, Z. et al. A foundation model for generalizable cancer diagnosis and survival prediction from histopathological images. Preprint at bioRxiv 
                https://doi.org/10.1101/2024.05.16.594499
                
               (2024)." href="https://www.nature.com/articles/#ref-CR17" id="ref-link-section-d5807050e520_6">17</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Hua, S., Yan, F., Shen, T., Ma, L. &amp; Zhang, X. PathoDuet: foundation models for pathological slide analysis of H&amp;E and IHC stains. Med. Image Anal. 97, 103289 (2024)." href="https://www.nature.com/articles/#ref-CR18" id="ref-link-section-d5807050e520_7">18</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Nechaev, D., Pchelnikov, A. &amp; Ivanova, E. Hibou: a family of foundational vision transformers for pathology. Preprint at arXiv 
                https://doi.org/10.48550/arXiv.2406.05074
                
               (2024)." href="https://www.nature.com/articles/#ref-CR19" id="ref-link-section-d5807050e520_8">19</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Lu, M. Y. et al. A visual-language foundation model for computational pathology. Nat. Med. 30, 863–874 (2024)." href="https://www.nature.com/articles/#ref-CR20" id="ref-link-section-d5807050e520_9">20</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Zimmermann, E. et al. Virchow2: scaling self-supervised mixed magnification models in pathology. Preprint at arXiv 
                https://doi.org/10.48550/arXiv.2408.00738
                
               (2024)." href="https://www.nature.com/articles/#ref-CR21" id="ref-link-section-d5807050e520_10">21</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Juyal, D. et al. PLUTO: pathology-universal transformer. Preprint at arXiv 
                https://doi.org/10.48550/arXiv.2405.07905
                
               (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR22" id="ref-link-section-d5807050e523">22</a></sup>. For example, the PLUTO foundation model<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Juyal, D. et al. PLUTO: pathology-universal transformer. Preprint at arXiv 
                https://doi.org/10.48550/arXiv.2405.07905
                
               (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR22" id="ref-link-section-d5807050e527">22</a></sup> has demonstrated superior performance relative to other CNN-based models in nuclei and gland segmentation, with F1 scores (which compute the relative contribution of precision and recall) of 0.8 for immunohistochemistry-based segmentation and 0.65 for haematoxylin and eosin-based segmentation.

Foundation models have shown potential for predicting patient prognosis and treatment response<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Chen, R. J. et al. Towards a general-purpose foundation model for computational pathology. Nat. Med. 30, 850–862 (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR11" id="ref-link-section-d5807050e534">11</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Vorontsov, E. A foundation model for clinical-grade computational pathology and rare cancers detection. Nat. Med. 30, 2924–2935 (2024)." href="https://www.nature.com/articles/#ref-CR13" id="ref-link-section-d5807050e537">13</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Dippel, J. et al. RudolfV: a foundation model by pathologists for pathologists. Preprint at arXiv 
                https://doi.org/10.48550/arXiv.2401.04079
                
               (2024)." href="https://www.nature.com/articles/#ref-CR14" id="ref-link-section-d5807050e537_1">14</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Wang, X. et al. A pathology foundation model for cancer diagnosis and prognosis prediction. Nature 634, 970–978 (2024)." href="https://www.nature.com/articles/#ref-CR15" id="ref-link-section-d5807050e537_2">15</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Ma, J. et al. Towards a generalizable pathology foundation model via unified knowledge distillation. Preprint at arXiv 
                https://doi.org/10.48550/arXiv.2407.18449
                
               (2024)." href="https://www.nature.com/articles/#ref-CR16" id="ref-link-section-d5807050e537_3">16</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Yang, Z. et al. A foundation model for generalizable cancer diagnosis and survival prediction from histopathological images. Preprint at bioRxiv 
                https://doi.org/10.1101/2024.05.16.594499
                
               (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR17" id="ref-link-section-d5807050e540">17</a></sup>. For example, the UNI foundation model<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Chen, R. J. et al. Towards a general-purpose foundation model for computational pathology. Nat. Med. 30, 850–862 (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR11" id="ref-link-section-d5807050e544">11</a></sup> has demonstrated considerable potential in characterizing the tumour microenvironment, including identifying tumour-infiltrating lymphocytes, which have a key role in prognosis and response prediction<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Li, H. et al. A prognostic and predictive computational pathology immune signature for ductal carcinoma in situ: retrospective results from a cohort within the UK/ANZ DCIS trial. Lancet Digital Health 6, e562–e569 (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR23" id="ref-link-section-d5807050e548">23</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 41" title="Corredor, G. et al. Spatial architecture and arrangement of tumor-infiltrating lymphocytes for predicting likelihood of recurrence in early-stage non-small cell lung cancer. Clin. Cancer Res. 25, 1526–1534 (2019)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR41" id="ref-link-section-d5807050e551">41</a></sup>. In a pan-cancer analysis using the tumour-infiltrating lymphocytes dataset from The Cancer Genome Atlas, the UNI model achieved a specimen-level area under the curve (AUC) of 0.979 for tumour-infiltrating lymphocyte detection, outperforming previous frameworks reporting AUCs ranging from 0.97 to 0.975 (refs. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Li, H. et al. A prognostic and predictive computational pathology immune signature for ductal carcinoma in situ: retrospective results from a cohort within the UK/ANZ DCIS trial. Lancet Digital Health 6, e562–e569 (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR23" id="ref-link-section-d5807050e555">23</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 41" title="Corredor, G. et al. Spatial architecture and arrangement of tumor-infiltrating lymphocytes for predicting likelihood of recurrence in early-stage non-small cell lung cancer. Clin. Cancer Res. 25, 1526–1534 (2019)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR41" id="ref-link-section-d5807050e558">41</a></sup>). Another important application is tumour detection and classification, in which these models effectively differentiate between benign and malignant lesions in pan-cancer datasets<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Vorontsov, E. A foundation model for clinical-grade computational pathology and rare cancers detection. Nat. Med. 30, 2924–2935 (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR13" id="ref-link-section-d5807050e562">13</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Zimmermann, E. et al. Virchow2: scaling self-supervised mixed magnification models in pathology. Preprint at arXiv 
                https://doi.org/10.48550/arXiv.2408.00738
                
               (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR21" id="ref-link-section-d5807050e565">21</a></sup>. The Virchow foundation model<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Vorontsov, E. A foundation model for clinical-grade computational pathology and rare cancers detection. Nat. Med. 30, 2924–2935 (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR13" id="ref-link-section-d5807050e570">13</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Zimmermann, E. et al. Virchow2: scaling self-supervised mixed magnification models in pathology. Preprint at arXiv 
                https://doi.org/10.48550/arXiv.2408.00738
                
               (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR21" id="ref-link-section-d5807050e573">21</a></sup>, for example, has set new benchmarks in computational pathology, surpassing previous frameworks in tasks such as pan-cancer detection and achieving a specimen-level AUC of 0.95 for common cancers (compared with a typical range of 0.9–0.93 in previous frameworks) and 0.93 for rare cancers (compared with a typical range of 0.88–0.92 in previous frameworks).

We must also acknowledge the limitations of these models. A comparative analysis<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 60" title="Alfasly, S. et al. Foundation models for histopathology – fanfare or flair. Mayo Clin. Proceedings: Digital Health 2, 165–174 (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR60" id="ref-link-section-d5807050e580">60</a></sup> showed that CNN-based models can outperform foundation models. This study specifically highlighted the superior performance of CNN-based models in whole-slide image retrieval (that is, identification of similarities with cancer-related features in the training dataset) for breast, liver, skin and colorectal cancer<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 60" title="Alfasly, S. et al. Foundation models for histopathology – fanfare or flair. Mayo Clin. Proceedings: Digital Health 2, 165–174 (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR60" id="ref-link-section-d5807050e584">60</a></sup>. This finding underscores the importance of a meticulous approach to curating high-quality, domain-specific datasets to train reliable and robust foundation models to carry out specific tasks with the highest possible level of accuracy.

The development of multimodal AI, which combines data from multiple sources, including pathology images and genomics, and data reported as text from clinical reports, is another substantial advance in computational pathology<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 50" title="Unger, M. &amp; Kather, J. N. A systematic analysis of deep learning in genomics and histopathology for precision oncology. BMC Med. Genomics 17, 48 (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR50" id="ref-link-section-d5807050e592">50</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Spratt, D. E. et al. Artificial intelligence predictive model for hormone therapy use in prostate cancer. NEJM Evid. 2, EVIDoa2300023 (2023)." href="https://www.nature.com/articles/#ref-CR61" id="ref-link-section-d5807050e595">61</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Boehm, K. M. et al. Multimodal data integration using machine learning improves risk stratification of high-grade serous ovarian cancer. Nat. Cancer 3, 723–733 (2022)." href="https://www.nature.com/articles/#ref-CR62" id="ref-link-section-d5807050e595_1">62</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Lu, C., Shiradkar, R. &amp; Liu, Z. Integrating pathomics with radiomics and genomics for cancer prognosis: a brief review. Chin. J. Cancer Res. 33, 563–573 (2021)." href="https://www.nature.com/articles/#ref-CR63" id="ref-link-section-d5807050e595_2">63</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Guo, J. et al. Cross-modal deep learning model for predicting pathologic complete response to neoadjuvant chemotherapy in breast cancer. NPJ Precis. Onc. 8, 189 (2024)." href="https://www.nature.com/articles/#ref-CR64" id="ref-link-section-d5807050e595_3">64</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 65" title="Goyal, M. et al. A multi-model approach integrating whole-slide imaging and clinicopathologic features to predict breast cancer recurrence risk. NPJ Breast Cancer 10, 93 (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR65" id="ref-link-section-d5807050e598">65</a></sup>. Multimodal approaches facilitate a more comprehensive understanding of disease mechanisms and patient outcomes. For example, Spratt et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 61" title="Spratt, D. E. et al. Artificial intelligence predictive model for hormone therapy use in prostate cancer. NEJM Evid. 2, EVIDoa2300023 (2023)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR61" id="ref-link-section-d5807050e602">61</a></sup> developed a multimodal AI approach using pretreatment prostate tissue images and clinical data (at a median follow-up duration of 10 years) from 5,727 patients enrolled in five phase III trials evaluating radiotherapy with or without androgen-deprivation therapy (ADT). The model predicted distant metastases and identified patients likely to benefit from ADT. Validation using data from the NRG Oncology/RTOG 9408 trial, with a median follow-up of 14.9 years, showed that ADT significantly reduced the occurrence of metastases in 34% of patients classified as model-positive, while providing no benefit to the 66% classified as model-negative<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 61" title="Spratt, D. E. et al. Artificial intelligence predictive model for hormone therapy use in prostate cancer. NEJM Evid. 2, EVIDoa2300023 (2023)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR61" id="ref-link-section-d5807050e606">61</a></sup>. This study highlights the predictive value of AI and its potential role in improving clinical decision-making.

## Advances in clinical AI deployment

### Regulatory decisions

Since 2019, the number of FDA approvals for devices based on AI and/or [machine learning](https://www.nature.com/articles/s41571-025-00991-6#Glos8) has increased substantially, rising from fewer than 70 to currently >1,000 (refs. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 66" title="Joshi, G. et al. FDA-approved artificial intelligence and machine learning (AI/ML)-enabled medical devices: an updated landscape. Electronics 13, 498 (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR66" id="ref-link-section-d5807050e625">66</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 67" title="FDA. Artificial intelligence and machine learning (AI/ML)-enabled medical devices. FDA 
                https://www.fda.gov/medical-devices/software-medical-device-samd/artificial-intelligence-and-machine-learning-aiml-enabled-medical-devices
                
               (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR67" id="ref-link-section-d5807050e628">67</a></sup>). This growth has prompted the evolution of regulatory frameworks, with the FDA and the European Union (EU) Conformité Européenne (CE) adapting guidelines to regulate AI-enabled medical devices<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 68" title="Cheng, J. Y., Abel, J. T., Balis, U. G. J., McClintock, D. S. &amp; Pantanowitz, L. Challenges in the development, deployment, and regulation of artificial intelligence in anatomic pathology. Am. J. Pathol. 191, 1684–1692 (2021)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR68" id="ref-link-section-d5807050e632">68</a></sup>. Notably, the Artificial Intelligence Act proposed by the EU in 2024 introduces a comprehensive legal framework, establishing EU-wide rules on data quality, transparency, human oversight and accountability through a risk-based approach<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 69" title="European Commission. Proposal for a Regulation of the European Parliament and of the Council Laying Down Harmonised Rules on Artificial Intelligence (Artificial Intelligence Act) and Amending Certain Union Legislative Acts (Publications Office of the European Union, 2021)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR69" id="ref-link-section-d5807050e636">69</a></sup>. In parallel to developing this regulatory framework, AI-based tools such as Concentriq AP-Dx<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 70" title="Proscia. Proscia receives FDA 510(k) clearance for Concentriq AP-Dx. Proscia 
                proscia.com/press-releases/proscia-receives-fda-510k-clearance-for-concentriq-ap-dx-2/
                
               (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR70" id="ref-link-section-d5807050e640">70</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 71" title="IBEX. AI-First Approach to Cancer Diagnostics Earns Ibex Medical Analytics Notable Milestones. IBEX 
                ibex-ai.com/milestones/
                
               (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR71" id="ref-link-section-d5807050e643">71</a></sup> and Galen Prostate<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 71" title="IBEX. AI-First Approach to Cancer Diagnostics Earns Ibex Medical Analytics Notable Milestones. IBEX 
                ibex-ai.com/milestones/
                
               (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR71" id="ref-link-section-d5807050e648">71</a></sup> obtained CE marking (under the EU’s In-vitro Diagnostic Medical Devices Regulation (IVDR))<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 72" title="Directorate-General for Health and Food Safety. Manufacturer IVD. ec.europa.eu 
                https://health.ec.europa.eu/medical-devices-new-regulations/getting-ready-new-regulations/manufacturer-ivd_en
                
               (accessed 7 February 2025)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR72" id="ref-link-section-d5807050e652">72</a></sup>.

In the USA, the FDA authorized a [De Novo Classification Request](https://www.nature.com/articles/s41571-025-00991-6#Glos10) for the AI-based prostate cancer detection tool Paige Prostate<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 73" title="Food and Drug Administration. FDA Authorizes Software that Can Help Identify Prostate Cancer. FDA 
                www.fda.gov/news-events/press-announcements/fda-authorizes-software-can-help-identify-prostate-cancer
                
               (2021)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR73" id="ref-link-section-d5807050e662">73</a></sup> in 2021, followed by [510(k) clearance](https://www.nature.com/articles/s41571-025-00991-6#Glos9) decisions for other AI-based pathology tools developed for diagnostic purposes such as PathAI, Roche Digital Pathology Dx<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 74" title="Roche Diagnostics. Roche Receives FDA Clearance on its Digital Pathology Solution for Diagnostic Use. Roche Diagnostics 
                diagnostics.roche.com/us/en/news-listing/2024/roche-receives-fda-clearance-on-its-digital-pathology-solution-for-diagnostic-use.html
                
               (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR74" id="ref-link-section-d5807050e669">74</a></sup> and Concentriq AP-Dx<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 70" title="Proscia. Proscia receives FDA 510(k) clearance for Concentriq AP-Dx. Proscia 
                proscia.com/press-releases/proscia-receives-fda-510k-clearance-for-concentriq-ap-dx-2/
                
               (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR70" id="ref-link-section-d5807050e673">70</a></sup>.

Nevertheless, approvals of AI-based tools for digital pathology are strikingly low relative to those for other medical applications. As of December 2024 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 67" title="FDA. Artificial intelligence and machine learning (AI/ML)-enabled medical devices. FDA 
                https://www.fda.gov/medical-devices/software-medical-device-samd/artificial-intelligence-and-machine-learning-aiml-enabled-medical-devices
                
               (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR67" id="ref-link-section-d5807050e680">67</a></sup>), radiology dominates the software-as-a-medical-device space in the USA by a substantial margin (76% of all FDA-approved devices), followed by cardiovascular disease (10%) and neurology (4%), while only three devices (<0.5%) have been approved for pathology (Fig. [1](https://www.nature.com/articles/s41571-025-00991-6#Fig1)). This trend reflects a slower than anticipated integration of AI in digital pathology, especially considering that approvals of radiology devices typically have shorter median approval wait times (126 days) than those in other specialties (including 193 days for pathology)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 66" title="Joshi, G. et al. FDA-approved artificial intelligence and machine learning (AI/ML)-enabled medical devices: an updated landscape. Electronics 13, 498 (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR66" id="ref-link-section-d5807050e687">66</a></sup>.

![figure 1](https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41571-025-00991-6/MediaObjects/41571_2025_991_Fig1_HTML.png)

**Fig. 1: FDA approvals of artificial intelligence-based or machine learning-based devices across medical specialties.**

### Reimbursement

#### General frameworks for in vitro diagnostic tools

In the USA, Current Procedural Terminology (CPT) codes provide a standardized framework for health-care professionals to accurately report the use of medical services and procedures, ensuring consistency and efficiency in claim processing, medical care review and insurance reimbursement<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 75" title="American Medical Association. CPT® Overview and Code Approval. AMA 
                www.ama-assn.org/practice-management/cpt/cpt-overview-and-code-approval
                
               (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR75" id="ref-link-section-d5807050e727">75</a></sup>. Interestingly, a wide range of CPT codes used for radiological procedures cover imaging services (codes 70010–76499)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 76" title="Centers for Medicare &amp; Medicaid Services. Medicare Claims Processing Manual. CMS.gov 
                www.cms.gov/Regulations-and-Guidance/Guidance/Manuals/Internet-Only-Manuals-IOMs-Items/CMS018912
                
               (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR76" id="ref-link-section-d5807050e731">76</a></sup>. Meanwhile, digital pathology remains in the nascent stage of integration into clinical practice. Before 2019, pathology-related CPT codes primarily covered conventional diagnostic services, such as tissue biopsies and microscopic examinations, without specific provisions for digital or computational pathology. In 2024, 30 new Category III add-on codes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 77" title="College of American Pathologists. 13 Digital Pathology Codes you Should Use in 2023 and Beyond. College of American Pathologists 
                www.cap.org/member-resources/articles/13-digital-pathology-codes-you-should-use-in-2023-and-beyond
                
               (2023)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR77" id="ref-link-section-d5807050e735">77</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 78" title="College of American Pathologists. Digital Pathology Codes. College of American Pathologists 
                www.cap.org/advocacy/payments-for-pathology-services/digital-pathology-codes
                
               (2023)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR78" id="ref-link-section-d5807050e738">78</a></sup>, which are temporary CPT codes used to track new technologies, were added to the previously existing 13 codes. These new codes were introduced for services involving digitization of glass slides, which currently have no established payment rates, and they are expected to become established CPT codes (of Category I)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 79" title="College of American Pathologists. How to Use the New Digital Pathology CPT Codes: CAP Answers Questions. College of American Pathologists 
                www.cap.org/member-resources/articles/how-to-use-the-new-digital-pathology-cpt-codes
                
               (2023)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR79" id="ref-link-section-d5807050e742">79</a></sup>.

In the EU, some public reimbursement systems support digital health initiatives<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 80" title="Mantovani, A., Leopaldi, C., Nighswander, C. M. &amp; Bidino, R. D. Access and reimbursement pathways for digital health solutions and in vitro diagnostic devices: current scenario and challenges. Front. Med. Technol. 5, 1101476 (2023)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR80" id="ref-link-section-d5807050e749">80</a></sup> with funding from various sources to establish digital pathology pipelines<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 66" title="Joshi, G. et al. FDA-approved artificial intelligence and machine learning (AI/ML)-enabled medical devices: an updated landscape. Electronics 13, 498 (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR66" id="ref-link-section-d5807050e753">66</a></sup>. For example, in Germany fast-tracked reimbursement pathways for digital health applications are progressing through the Digital Health Applications (DiGA) programme<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 80" title="Mantovani, A., Leopaldi, C., Nighswander, C. M. &amp; Bidino, R. D. Access and reimbursement pathways for digital health solutions and in vitro diagnostic devices: current scenario and challenges. Front. Med. Technol. 5, 1101476 (2023)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR80" id="ref-link-section-d5807050e757">80</a></sup>. This programme provides a fast-track process for regulatory approval and reimbursement, allowing clinicians to prescribe approved digital health applications that are fully reimbursable by statutory health insurance, covering a large population<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 80" title="Mantovani, A., Leopaldi, C., Nighswander, C. M. &amp; Bidino, R. D. Access and reimbursement pathways for digital health solutions and in vitro diagnostic devices: current scenario and challenges. Front. Med. Technol. 5, 1101476 (2023)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR80" id="ref-link-section-d5807050e761">80</a></sup>. In France, through the PECAN programme, similar avenues are being explored for digital health solutions, including telemonitoring and digital therapeutics<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 80" title="Mantovani, A., Leopaldi, C., Nighswander, C. M. &amp; Bidino, R. D. Access and reimbursement pathways for digital health solutions and in vitro diagnostic devices: current scenario and challenges. Front. Med. Technol. 5, 1101476 (2023)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR80" id="ref-link-section-d5807050e765">80</a></sup>. Many European nations, such as the UK, Netherlands and Belgium, also reimburse AI-based in vitro diagnostic tests with funds from their national health-care reimbursement schemes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 80" title="Mantovani, A., Leopaldi, C., Nighswander, C. M. &amp; Bidino, R. D. Access and reimbursement pathways for digital health solutions and in vitro diagnostic devices: current scenario and challenges. Front. Med. Technol. 5, 1101476 (2023)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR80" id="ref-link-section-d5807050e770">80</a></sup>.

In Asia, Japan<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 81" title="Pharmaceuticals and Medical Devices Agency. The 1st Subcommittee on Software as a Medical Device Utilizing AI and Machine Learning. PMDA 
                www.pmda.go.jp/english/rs-sb-std/sb/subcommittees/0024.html
                
               (2021)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR81" id="ref-link-section-d5807050e777">81</a></sup> and South Korea<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 82" title="Ministry of Food and Drug Safety. Guidance on the Review and Approval of Artificial Intelligence (AI)-based Medical Devices. Ministry of Food and Drug Safety 
                www.mfds.go.kr/eng/brd/m_40/view.do?seq=72627&amp;utm_source=chatgpt.com
                
               (2023)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR82" id="ref-link-section-d5807050e781">82</a></sup> have established regulatory pathways for AI-based or machine learning-based medical devices, emphasizing safety, transparency, and performance. Other nations, including China<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 83" title="National Medical Products Administration. NMPA Announcement on Guidance for the Classification Defining of AI-Based Medical Software Products. NMPA 
                english.nmpa.gov.cn/2021-07/08/c_660267.htm?utm_source=chatgpt.com
                
               (2021)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR83" id="ref-link-section-d5807050e785">83</a></sup> and India<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 84" title="Indian Council of Medical Research. Ethical Guidelines for Application of Artificial Intelligence in Biomedical Research and Healthcare. ICMR 
                www.icmr.gov.in/ethical-guidelines-for-application-of-artificial-intelligence-in-biomedical-research-and-healthcare
                
               (2023)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR84" id="ref-link-section-d5807050e789">84</a></sup>, have introduced preliminary guidelines aimed at addressing challenges associated with AI technologies within their health-care systems.

#### LDTs and in-house devices

Reimbursement frameworks are evolving to accommodate emerging technologies in digital pathology. Among these, LDTs and in-house devices are having a crucial role in addressing specific diagnostic needs within individual laboratories<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 85" title="Larsen, M. P. &amp; Tsang, P. Laboratory Developed Tests. PathologyOutlines.com 
                www.pathologyoutlines.com/topic/managementlabdevelopedtests.html
                
               (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR85" id="ref-link-section-d5807050e801">85</a></sup>. Originally, LDTs were low-risk, specialized assays developed in the USA, created for specific, local patient populations and used in small volumes. These tests were tailored to address unique clinical needs, and thus they were exempt from rigorous FDA oversight and operated under the Clinical Laboratory Improvement Amendments (CLIA) certification framework. However, technological advances and shifts in business practices, combined with the ability to rapidly transport patient samples nationwide, have led to a broader use of LDTs. These tests are now applied to a larger, more diverse population, with major laboratories processing specimens from a wide geographical area<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 86" title="Food and Drug Administration. Laboratory Developed Tests. FDA 
                www.fda.gov/medical-devices/in-vitro-diagnostics/laboratory-developed-tests
                
               (2025)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR86" id="ref-link-section-d5807050e805">86</a></sup>. Their use is covered by a type of CPT code referred to as Proprietary Laboratory Analyses (PLA) codes. For example, the ArteraAI Prostate Test<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 87" title="Artera. For Patients. Artera 
                https://artera.ai/for-patients
                
               (2025)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR87" id="ref-link-section-d5807050e809">87</a></sup> is an LDT available through a CLIA-certified laboratory designed to help clinicians to identify patients with localized prostate cancer who might benefit from ADT. This test has the unique PLA code 0376U<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 88" title="Codify. CPT® Code 0376U, Under Proprietary Laboratory Analyses. AAPC 
                www.aapc.com/codes/cpt-codes/0376U
                
               (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR88" id="ref-link-section-d5807050e813">88</a></sup>. The inclusion of the ArteraAI Prostate Test in the US National Comprehensive Cancer Network (NCCN) guidelines for prostate cancer management from 2024 with a category 2A recommendation (indicating uniform consensus based on low-level evidence) emphasizes the growing relevance of AI-enabled tools for personalized treatment<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 89" title="Schaeffer, E. M. et al. NCCN guidelines® insights: prostate cancer, version 3.2024. J. Natl Compr. Cancer Netw. 22, 140–150 (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR89" id="ref-link-section-d5807050e817">89</a></sup>. Yet, concerns about the reliability of LDTs persist. In 2015, the FDA published 20 case studies detailing instances in which LDTs produced inaccurate results that led to inappropriate or delayed treatments, particularly in conditions such as cancer and heart disease<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 86" title="Food and Drug Administration. Laboratory Developed Tests. FDA 
                www.fda.gov/medical-devices/in-vitro-diagnostics/laboratory-developed-tests
                
               (2025)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR86" id="ref-link-section-d5807050e822">86</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 90" title="Backman, I. The FDA’s Proposed Ruling on Lab Tests Could Have Unintended Consequences. Yale School of Medicine 
                medicine.yale.edu/news-article/the-fdas-proposed-ruling-on-lab-tests-could-have-unintended-consequences/
                
               (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR90" id="ref-link-section-d5807050e825">90</a></sup>. In response to these concerns, the FDA proposed new regulations in 2023 for LDTs to have the same stringent safety and efficacy standards as other medical tests, aiming to enhance patient protection and ensuring the reliability of LDTs<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 86" title="Food and Drug Administration. Laboratory Developed Tests. FDA 
                www.fda.gov/medical-devices/in-vitro-diagnostics/laboratory-developed-tests
                
               (2025)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR86" id="ref-link-section-d5807050e829">86</a></sup>. However, some experts have argued that this shift could lead to delays in testing for rare diseases and in responding to future health crises because laboratories developing tests with this purpose would first need to navigate the lengthy and costly FDA approval process<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 90" title="Backman, I. The FDA’s Proposed Ruling on Lab Tests Could Have Unintended Consequences. Yale School of Medicine 
                medicine.yale.edu/news-article/the-fdas-proposed-ruling-on-lab-tests-could-have-unintended-consequences/
                
               (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR90" id="ref-link-section-d5807050e833">90</a></sup>.

The EU’s IVDR framework for in-house devices is similar to the FDA’s approach to LDT regulation. The IVDR framework allows health-care institutions to develop and use in-house devices internally under specific conditions, ensuring safety and reliability<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 91" title="Dombrink, I. et al. Critical implications of IVDR for innovation in diagnostics: input from the Biomed Alliance Diagnostics Task Force. HemaSphere 6, e724 (2022)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR91" id="ref-link-section-d5807050e840">91</a></sup>. The main difference between the EU and US approaches is that Article 5 of the IVDR imposes more stringent requirements on in-house devices. These requirements include proving that no CE-marked equivalent exists or that the in-house device addresses specific patient needs that CE-marked in vitro diagnostic tests cannot meet<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 91" title="Dombrink, I. et al. Critical implications of IVDR for innovation in diagnostics: input from the Biomed Alliance Diagnostics Task Force. HemaSphere 6, e724 (2022)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR91" id="ref-link-section-d5807050e844">91</a></sup>. Additionally, in-house devices must be used only within the owner institution and cannot be transferred to other entities<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 91" title="Dombrink, I. et al. Critical implications of IVDR for innovation in diagnostics: input from the Biomed Alliance Diagnostics Task Force. HemaSphere 6, e724 (2022)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR91" id="ref-link-section-d5807050e848">91</a></sup>. This approach enables European laboratories to continue innovative diagnostic practices while gradually harmonizing with regulatory expectations for safety and performance<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 91" title="Dombrink, I. et al. Critical implications of IVDR for innovation in diagnostics: input from the Biomed Alliance Diagnostics Task Force. HemaSphere 6, e724 (2022)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR91" id="ref-link-section-d5807050e852">91</a></sup>.

## Clinical deployment

The use of AI in digital pathology requires adherence to regulatory standards and compatibility with existing laboratory information systems for integration into clinical practice<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 92" title="Yousif, M., Hassell, L. &amp; Pantanowitz, L. in Digital Innovation for Healthcare in COVID-19 Pandemic (eds de Pablos, P. O. et al.) Ch. 7, 95–107 (Academic Press, 2022)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR92" id="ref-link-section-d5807050e866">92</a></sup>. A substantial proportion of the AI-based models that had been approved by the FDA as of 31 May 2023 for oncology and oncology-related fields (including digital pathology and cancer radiology) are for solid malignancies in general (34%), whereas 31% are focused on breast cancer and 8.5% each on prostate and lung cancers<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 93" title="Luchini, C., Pea, A. &amp; Scarpa, A. Artificial intelligence in oncology: current applications and future perspectives. Br. J. Cancer 126, 4–9 (2022)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR93" id="ref-link-section-d5807050e870">93</a></sup>. Most of the AI-based digital pathology tools currently used in oncology are focused on tasks such as quantification and identifying specific regions of interest in immunohistochemistry slides. AI-based models are also being widely applied to detect metastases in lymph nodes across multiple solid malignancies, including colorectal, breast and gastric cancers<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Wu, S. et al. An artificial intelligence model for detecting pathological lymph node metastasis in prostate cancer using whole slide images: a retrospective, multicentre, diagnostic study. eClinicalMedicine 71, 102580 (2024)." href="https://www.nature.com/articles/#ref-CR94" id="ref-link-section-d5807050e874">94</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Reddy, S., Shaheed, A., Seo, Y. &amp; Patel, R. Development of an artificial intelligence model for the classification of gastric carcinoma stages using pathology slides. Cureus 16, e56740 (2024)." href="https://www.nature.com/articles/#ref-CR95" id="ref-link-section-d5807050e874_1">95</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 96" title="van Dooijeweert, C. et al. Clinical implementation of artificial-intelligence-assisted detection of breast cancer metastases in sentinel lymph nodes: the CONFIDENT-B single-center, non-randomized clinical trial. Nat. Cancer 5, 1195–1205 (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR96" id="ref-link-section-d5807050e877">96</a></sup>.

Wider clinical adoption of these tools will require independent validation and formal regulatory approvals. In 2019, the clinical deployment of AI tools in digital pathology was in its early stages and most of the institutions that were beginning to digitize images from biopsy samples used them for teaching or molecular tumour board purposes. A few advanced laboratories, however, had already begun scanning the entire institutional caseloads<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 97" title="Eloy, C. et al. Artificial intelligence-assisted cancer diagnosis improves the efficiency of pathologists in prostatic biopsies. Virchows Arch. 482, 595–604 (2023)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR97" id="ref-link-section-d5807050e884">97</a></sup>, and this practice became more widely adopted during the COVID-19 pandemic to facilitate work-from-home setups<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 92" title="Yousif, M., Hassell, L. &amp; Pantanowitz, L. in Digital Innovation for Healthcare in COVID-19 Pandemic (eds de Pablos, P. O. et al.) Ch. 7, 95–107 (Academic Press, 2022)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR92" id="ref-link-section-d5807050e888">92</a></sup>. Despite these developments, the global adoption of digital pathology has been slow, largely owing to infrastructural demands, including substantial investments in hardware (scanners), digital storage, powerful servers and infrastructure, but also owing to the need to gain support from pathologists<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 92" title="Yousif, M., Hassell, L. &amp; Pantanowitz, L. in Digital Innovation for Healthcare in COVID-19 Pandemic (eds de Pablos, P. O. et al.) Ch. 7, 95–107 (Academic Press, 2022)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR92" id="ref-link-section-d5807050e892">92</a></sup>. This slow adoption persists even though surveys conducted in several countries have shown encouraging trends. For example, a national survey conducted in Switzerland in 2019 revealed that 66% of pathologists would be comfortable making primary diagnoses on digitized slides, although the survey completion rate was only 39.5% (potentially indicating a selection bias)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 98" title="Dawson, H. Digital pathology – rising to the challenge. Front. Med. 9, 888896 (2022)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR98" id="ref-link-section-d5807050e896">98</a></sup>. Similarly, a survey in the UK found that 31% of institutions were using digital slides for primary diagnosis, with many respondents prioritizing digital pathology owing to its potential to improve efficiency and collaboration<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 99" title="Williams, B. J., Lee, J., Oien, K. A. &amp; Treanor, D. Digital pathology access and usage in the UK: results from a national survey on behalf of the National Cancer Research Institute’s CM-Path initiative. J. Clin. Pathol. 71, 463–466 (2018)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR99" id="ref-link-section-d5807050e900">99</a></sup>.

Validation in prospective clinical trials is the way forward for the clinical adoption of AI-based tools in digital pathology (Fig. [2](https://www.nature.com/articles/s41571-025-00991-6#Fig2)). For example, ongoing trials involving patients with colorectal or non-small-cell lung cancer are using digital image analyses to identify potential immune cell biomarkers, such as cytotoxic T cells, PD-L1 expression, tumour-infiltrating lymphocyte density and spatial relationships of immune cells within the tumour microenvironment<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Baxi, V., Edwards, R., Montalto, M. &amp; Saha, S. Digital pathology and artificial intelligence in translational medicine and clinical practice. Mod. Pathol. 35, 23–32 (2022)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR6" id="ref-link-section-d5807050e910">6</a></sup>. An ongoing prospective observational study is using an automated tool to identify tumour areas in whole-slide images from individuals with intestinal metaplasia, thus addressing the shortage of pathologists available for gastric cancer screening (NCT05447221). The DPAILO-1 trial (NCT06344364) is investigating the prognostic performance of a digital pathology score for predicting hepatic decompensation in patients with metabolic dysfunction-associated steatohepatitis, which can lead to hepatocellular carcinoma. Furthermore, AI-assisted image biomarker analyses have been used to retrospectively re-evaluate samples from the UK/ANZ DCIS trial<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Li, H. et al. A prognostic and predictive computational pathology immune signature for ductal carcinoma in situ: retrospective results from a cohort within the UK/ANZ DCIS trial. Lancet Digital Health 6, e562–e569 (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR23" id="ref-link-section-d5807050e914">23</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 100" title="Aggarwal, A., Badve, S. S., Cuzick, J., Madabhushi, A. &amp; Thorat, M. A. A computational pathology collagen signature predictive of tamoxifen benefit in ductal carcinoma in situ: results from a cohort within the UK/ANZ DCIS randomized trial [abstract 171P]. Ann. Oncol. 35 (Suppl. 2), S284 (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR100" id="ref-link-section-d5807050e917">100</a></sup> and six NRG Radiation Therapy Oncology Group trials<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 101" title="Spratt, D. E. et al. Patient-level data meta-analysis of a multi-modal artificial intelligence (MMAI) prognostic biomarker in high-risk prostate cancer: results from six NRG/RTOG phase III randomized trials [abstract]. J. Clin. Oncol. 41 (Suppl. 6), 299 (2023)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR101" id="ref-link-section-d5807050e921">101</a></sup>, involving patients with ductal carcinoma in situ of the breast and high-risk prostate cancer, respectively. For example, Aggarwal et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 100" title="Aggarwal, A., Badve, S. S., Cuzick, J., Madabhushi, A. &amp; Thorat, M. A. A computational pathology collagen signature predictive of tamoxifen benefit in ductal carcinoma in situ: results from a cohort within the UK/ANZ DCIS randomized trial [abstract 171P]. Ann. Oncol. 35 (Suppl. 2), S284 (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR100" id="ref-link-section-d5807050e925">100</a></sup> developed a collagen-based computational pathology biomarker and demonstrated its role in predicting tamoxifen benefit in the UK/ANZ DCIS trial (*n* = 755 patients). These analyses revealed granular patient stratifications and have potential to refine treatment strategies. These initiatives demonstrate the potential of AI-based digital pathology not only for retrospective analyses but also to automate and streamline prospective clinical trials, making drug development more efficient and cost-effective<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 102" title="Vora, L. K. et al. Artificial intelligence in pharmaceutical technology and drug delivery design. Pharmaceutics 15, 1916 (2023)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR102" id="ref-link-section-d5807050e933">102</a></sup>. Despite the increasing efforts to incorporate AI into digital pathology, especially within prospective clinical trials, these initiatives remain limited in both scope and number<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 103" title="Lam, T. Y. T. et al. Randomized controlled trials of artificial intelligence in clinical practice: systematic review. J. Med. Internet Res. 24, e37188 (2022)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR103" id="ref-link-section-d5807050e937">103</a></sup>.

![figure 2](https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41571-025-00991-6/MediaObjects/41571_2025_991_Fig2_HTML.png)

**Fig. 2: Proposed strategy for achieving higher levels of evidence in the validation of artificial intelligence-based or machine learning-based models for digital pathology applications.**

## Opportunities and challenges

Digital pathology has transformed the way pathology slides are viewed and analysed<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 104" title="Kiran, N. et al. Digital pathology: transforming diagnosis in the digital age. Cureus 15, e44620 (2023)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR104" id="ref-link-section-d5807050e985">104</a></sup>. The introduction of the first whole slide scanner in 1999 constituted a substantial advance in the digitization of pathology images<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 105" title="Zarella, M. D. et al. A practical guide to whole slide imaging: a white paper from the digital pathology association. Arch. Pathol. Laboratory Med. 143, 222–234 (2019)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR105" id="ref-link-section-d5807050e989">105</a></sup>. However, digital pathology only began gaining traction in the 2010s, with whole-slide imaging facilitating more widespread adoption<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 106" title="Pantanowitz, L. Digital images and the future of digital pathology. J. Pathol. Inform. 1, 15 (2010)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR106" id="ref-link-section-d5807050e993">106</a></sup>. In just over two decades since the commercialization of the first slide scanner, and ~12 years since the initial implementation of AI models in pathology, computational pathologists have made substantial strides in the development and deployment of these algorithms.

Computational pathology has evolved considerably in the past 5 years, from the early use of CNNs<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="O’Shea, K. &amp; Nash, R. An introduction to convolutional neural networks. Preprint at arXiv 
                https://doi.org/10.48550/arXiv.1511.08458
                
               (2015)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR8" id="ref-link-section-d5807050e1000">8</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Coudray, N. et al. Classification and mutation prediction from non–small cell lung cancer histopathology images using deep learning. Nat. Med. 24, 1559–1567 (2018)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR9" id="ref-link-section-d5807050e1003">9</a></sup> to the integration of advanced technologies, such as foundation models<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Chen, R. J. et al. Towards a general-purpose foundation model for computational pathology. Nat. Med. 30, 850–862 (2024)." href="https://www.nature.com/articles/#ref-CR11" id="ref-link-section-d5807050e1007">11</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Xu, H. et al. A whole-slide foundation model for digital pathology from real-world data. Nature 630, 181–188 (2024)." href="https://www.nature.com/articles/#ref-CR12" id="ref-link-section-d5807050e1007_1">12</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Vorontsov, E. A foundation model for clinical-grade computational pathology and rare cancers detection. Nat. Med. 30, 2924–2935 (2024)." href="https://www.nature.com/articles/#ref-CR13" id="ref-link-section-d5807050e1007_2">13</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Dippel, J. et al. RudolfV: a foundation model by pathologists for pathologists. Preprint at arXiv 
                https://doi.org/10.48550/arXiv.2401.04079
                
               (2024)." href="https://www.nature.com/articles/#ref-CR14" id="ref-link-section-d5807050e1007_3">14</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Wang, X. et al. A pathology foundation model for cancer diagnosis and prognosis prediction. Nature 634, 970–978 (2024)." href="https://www.nature.com/articles/#ref-CR15" id="ref-link-section-d5807050e1007_4">15</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Ma, J. et al. Towards a generalizable pathology foundation model via unified knowledge distillation. Preprint at arXiv 
                https://doi.org/10.48550/arXiv.2407.18449
                
               (2024)." href="https://www.nature.com/articles/#ref-CR16" id="ref-link-section-d5807050e1007_5">16</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Yang, Z. et al. A foundation model for generalizable cancer diagnosis and survival prediction from histopathological images. Preprint at bioRxiv 
                https://doi.org/10.1101/2024.05.16.594499
                
               (2024)." href="https://www.nature.com/articles/#ref-CR17" id="ref-link-section-d5807050e1007_6">17</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Hua, S., Yan, F., Shen, T., Ma, L. &amp; Zhang, X. PathoDuet: foundation models for pathological slide analysis of H&amp;E and IHC stains. Med. Image Anal. 97, 103289 (2024)." href="https://www.nature.com/articles/#ref-CR18" id="ref-link-section-d5807050e1007_7">18</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Nechaev, D., Pchelnikov, A. &amp; Ivanova, E. Hibou: a family of foundational vision transformers for pathology. Preprint at arXiv 
                https://doi.org/10.48550/arXiv.2406.05074
                
               (2024)." href="https://www.nature.com/articles/#ref-CR19" id="ref-link-section-d5807050e1007_8">19</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Lu, M. Y. et al. A visual-language foundation model for computational pathology. Nat. Med. 30, 863–874 (2024)." href="https://www.nature.com/articles/#ref-CR20" id="ref-link-section-d5807050e1007_9">20</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Zimmermann, E. et al. Virchow2: scaling self-supervised mixed magnification models in pathology. Preprint at arXiv 
                https://doi.org/10.48550/arXiv.2408.00738
                
               (2024)." href="https://www.nature.com/articles/#ref-CR21" id="ref-link-section-d5807050e1007_10">21</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Juyal, D. et al. PLUTO: pathology-universal transformer. Preprint at arXiv 
                https://doi.org/10.48550/arXiv.2405.07905
                
               (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR22" id="ref-link-section-d5807050e1010">22</a></sup>, for various applications. The focus has progressively shifted from basic tasks, such as image classification, segmentation and detection, to more complex applications, including predicting patient prognosis and response to treatment<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Chen, R. J. et al. Towards a general-purpose foundation model for computational pathology. Nat. Med. 30, 850–862 (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR11" id="ref-link-section-d5807050e1014">11</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Wang, X. et al. A pathology foundation model for cancer diagnosis and prognosis prediction. Nature 634, 970–978 (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR15" id="ref-link-section-d5807050e1017">15</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Zimmermann, E. et al. Virchow2: scaling self-supervised mixed magnification models in pathology. Preprint at arXiv 
                https://doi.org/10.48550/arXiv.2408.00738
                
               (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR21" id="ref-link-section-d5807050e1020">21</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Juyal, D. et al. PLUTO: pathology-universal transformer. Preprint at arXiv 
                https://doi.org/10.48550/arXiv.2405.07905
                
               (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR22" id="ref-link-section-d5807050e1023">22</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Fashi, P. A., Hemati, S., Babaie, M., Gonzalez, R. &amp; Tizhoosh, H. R. A self-supervised contrastive learning approach for whole slide image representation in digital pathology. J. Pathol. Inf. 13, 100133 (2022)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR36" id="ref-link-section-d5807050e1026">36</a></sup>. This transition has occurred in parallel with a remarkable growth in academic research in digital pathology, with ~10,000 AI-related and computational pathology-related pathology articles indexed in PubMed over the past 5 years compared with ~3,000 publications between 2015 and 2019 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 107" title="Morales, S., Engan, K. &amp; Naranjo, V. Artificial intelligence in computational pathology – challenges and future directions. Digital Signal. Process. 119, 103196 (2021)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR107" id="ref-link-section-d5807050e1030">107</a></sup>). Alongside this expansion, the digital pathology industry has attracted substantial investment interest, with cumulative funding reaching US $1.76 billion<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 108" title="Fitt, I. Digital Pathology – World – 2023. Signify Research 
                www.signifyresearch.net/market-intelligence/digital-pathology-world-2023/
                
               (2023)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR108" id="ref-link-section-d5807050e1034">108</a></sup>. Notably, three companies (Owkin, PathAI and Paige) have collectively raised over $780 million<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 108" title="Fitt, I. Digital Pathology – World – 2023. Signify Research 
                www.signifyresearch.net/market-intelligence/digital-pathology-world-2023/
                
               (2023)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR108" id="ref-link-section-d5807050e1039">108</a></sup>. Major pharmaceutical firms have also established specialized teams providing end-to-end digital pathology solutions<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 109" title="Law, R. Roche gains 510(k) clearance for digital pathology solution for diagnostic use. Medical Device Network 
                https://www.medicaldevice-network.com/news/roche-gains-501-clearance-for-digital-pathology-solution-for-diagnostic-use/?cf-view
                
               (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR109" id="ref-link-section-d5807050e1043">109</a></sup>, including AI-powered biomarker discovery for precision diagnostics<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 110" title="Abbvie. Precision Medicine. Abbvie 
                www.abbvie.com/science/areas-of-innovation/precision-medicine.html
                
               (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR110" id="ref-link-section-d5807050e1047">110</a></sup>.

Despite these developments, a gap persists between academic research, industry developers and clinical implementation of AI-based applications for digital pathology, as demonstrated by the limited number of FDA-approved AI tools for digital pathology<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 97" title="Eloy, C. et al. Artificial intelligence-assisted cancer diagnosis improves the efficiency of pathologists in prostatic biopsies. Virchows Arch. 482, 595–604 (2023)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR97" id="ref-link-section-d5807050e1054">97</a></sup> (Fig. [1](https://www.nature.com/articles/s41571-025-00991-6#Fig1)). By contrast, radiology has seen a notable increase in approvals, with a faster median processing time of approximately 200 days compared with the 300-day median for pathology. This discrepancy could be attributed to the substantial amount of data generated and available in routine radiology that pathology has yet to adopt<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 66" title="Joshi, G. et al. FDA-approved artificial intelligence and machine learning (AI/ML)-enabled medical devices: an updated landscape. Electronics 13, 498 (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR66" id="ref-link-section-d5807050e1061">66</a></sup>. Indeed, pathology has historically relied on physical glass slides and manual interpretation, slowing the development of digital archives and computational tools<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 111" title="Berbís, M. Á. Digital and computational pathology: what a time to be alive! Mayo Clin. Proc. Digital Health 2, 18–20 (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR111" id="ref-link-section-d5807050e1065">111</a></sup>. Additionally, studies have demonstrated that digital pathology can lead to considerable cost and time savings by optimizing efficiency across various workflows<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 112" title="García-Rivello, H. et al. Digital pathology in Latin America. Surg. Exp. Pathol. 6, 10 (2023)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR112" id="ref-link-section-d5807050e1069">112</a></sup>. For example, a large analysis undertaken by a large regional pathology laboratory in the Netherlands<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 113" title="Baidoshvili, A. et al. Evaluating the benefits of digital pathology implementation: time savings in laboratory logistics. Histopathology 73, 784–794 (2018)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR113" id="ref-link-section-d5807050e1074">113</a></sup> reported average savings of 19 h per day across the institution, which resulted in an estimated annual saving of 120,000 €. A similar study at a US centre<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 114" title="Hanna, M. G. et al. Implementation of digital pathology offers clinical and operational increase in efficiency and cost savings. Arch. Pathol. Lab. Med. 143, 1545–1555 (2019)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR114" id="ref-link-section-d5807050e1078">114</a></sup> projected that the implementation of digital pathology workflows would result in a 5-year savings of $1.3 million. In an Italian institution<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 115" title="Vergani, A., Regis, B., Jocollé, G., Patetta, R. &amp; Rossi, G. Noninferiority diagnostic value, but also economic and turnaround time advantages from digital pathology. Am. J. Surg. Pathol. 42, 841 (2018)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR115" id="ref-link-section-d5807050e1082">115</a></sup>, costs per consultation dropped from 3,365 € using conventional methods to 300 € with the adoption of digital systems, and the turnaround time decreased from 12 days to 1.4 days on average. A similar trend was seen with the introduction of AI, which reduced processing time owing to automated case triage and intelligent case distribution (reviewed elsewhere<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 116" title="Zarella, M. D. et al. Artificial intelligence and digital pathology: clinical promise and deployment considerations. J. Med. Imaging 10, 051802 (2023)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR116" id="ref-link-section-d5807050e1086">116</a></sup>).

The implementation of digital pathology varies substantially across countries<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 117" title="Pinto, D. G., Bychkov, A., Tsuyama, N., Fukuoka, J. &amp; Eloy, C. Real-world implementation of digital pathology: results from an intercontinental survey. Lab. Invest. 103, 100261 (2023)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR117" id="ref-link-section-d5807050e1093">117</a></sup>. The results of a survey conducted across 127 pathology laboratories worldwide<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 117" title="Pinto, D. G., Bychkov, A., Tsuyama, N., Fukuoka, J. &amp; Eloy, C. Real-world implementation of digital pathology: results from an intercontinental survey. Lab. Invest. 103, 100261 (2023)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR117" id="ref-link-section-d5807050e1097">117</a></sup> revealed diverse challenges and approaches influenced by regional factors. European respondents focused on storage planning, laboratory information system integration and substantial upfront investments as key requirements for a smooth transition to full digitization<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 117" title="Pinto, D. G., Bychkov, A., Tsuyama, N., Fukuoka, J. &amp; Eloy, C. Real-world implementation of digital pathology: results from an intercontinental survey. Lab. Invest. 103, 100261 (2023)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR117" id="ref-link-section-d5807050e1101">117</a></sup>, whereas Asian respondents reported a more gradual implementation approach, probably shaped by institutional readiness and funding availability<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 117" title="Pinto, D. G., Bychkov, A., Tsuyama, N., Fukuoka, J. &amp; Eloy, C. Real-world implementation of digital pathology: results from an intercontinental survey. Lab. Invest. 103, 100261 (2023)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR117" id="ref-link-section-d5807050e1105">117</a></sup>. This study emphasizes that lowering equipment costs and securing funding are crucial for accelerating the adoption of digital pathology, with costs varying widely across countries owing to differences in institutional readiness, available resources and health-care systems<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 117" title="Pinto, D. G., Bychkov, A., Tsuyama, N., Fukuoka, J. &amp; Eloy, C. Real-world implementation of digital pathology: results from an intercontinental survey. Lab. Invest. 103, 100261 (2023)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR117" id="ref-link-section-d5807050e1109">117</a></sup>. Similarly, in the USA, a 2019 survey of Alzheimer’s Disease Research Centers found that 50% of scanner purchases and operational costs were supported by institutional funding, further emphasizing the reliance on institutional resources<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 118" title="Scalco, R. et al. The status of digital pathology and associated infrastructure within Alzheimer’s disease centers. J. Neuropathol. Exp. Neurol. 82, 202–211 (2023)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR118" id="ref-link-section-d5807050e1114">118</a></sup>. In a separate study, researchers based in India reported that ensuring a continuous power supply for overnight scanning, maintaining uninterrupted internet access and the need for high-resolution monitors are major challenges for the implementation of digital pathology in low-income and middle-income countries<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 119" title="Indu, M., Rathy, R. &amp; Binu, M. P. “Slide less pathology”: fairy tale or reality? J. Oral. Maxillofac. Pathol. 20, 284 (2016)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR119" id="ref-link-section-d5807050e1118">119</a></sup>. The emergence of competing technologies, such as light-sheet fluorescence microscopy (a technique that illuminates samples with a thin sheet of light for high-resolution, 3D imaging) and molecular digital pathology, offers slide-less solutions<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 120" title="Bhargava, R. &amp; Madabhushi, A. Emerging themes in image informatics and molecular analysis for digital pathology. Annu. Rev. Biomed. Eng. 18, 387–412 (2016)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR120" id="ref-link-section-d5807050e1122">120</a></sup>. Unlike traditional glass slide-based approaches, these methods enable non-destructive, volumetric and chemical imaging (a label-free imaging method that uses the intrinsic chemical properties of tissues, such as vibrational or spectroscopic signals, to generate detailed molecular and structural information about a specimen), providing molecular markers of tissue specimens directly rather than requiring the digitization of glass slides<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 120" title="Bhargava, R. &amp; Madabhushi, A. Emerging themes in image informatics and molecular analysis for digital pathology. Annu. Rev. Biomed. Eng. 18, 387–412 (2016)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR120" id="ref-link-section-d5807050e1126">120</a></sup>. Light-sheet fluorescence microscopy, for example, enables 3D visualization without the need for sectioning<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 121" title="Liu, Y., Levenson, R. M. &amp; Jenkins, M. W. Slide over: advances in slide-free optical microscopy as drivers of diagnostic pathology. Am. J. Pathol. 192, 180–194 (2022)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR121" id="ref-link-section-d5807050e1130">121</a></sup>, whereas chemical imaging techniques<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 120" title="Bhargava, R. &amp; Madabhushi, A. Emerging themes in image informatics and molecular analysis for digital pathology. Annu. Rev. Biomed. Eng. 18, 387–412 (2016)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR120" id="ref-link-section-d5807050e1134">120</a></sup> offer molecular specificity without staining. These innovations have the potential to transform traditional workflows by providing more detailed and comprehensive information on tissue structure and biochemical characteristics. The adoption of these approaches could lead to changing and adapting AI technologies as newer digital pathology platforms are introduced.

Although some digital pathology scanners have received regulatory approval<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 70" title="Proscia. Proscia receives FDA 510(k) clearance for Concentriq AP-Dx. Proscia 
                proscia.com/press-releases/proscia-receives-fda-510k-clearance-for-concentriq-ap-dx-2/
                
               (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR70" id="ref-link-section-d5807050e1142">70</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 74" title="Roche Diagnostics. Roche Receives FDA Clearance on its Digital Pathology Solution for Diagnostic Use. Roche Diagnostics 
                diagnostics.roche.com/us/en/news-listing/2024/roche-receives-fda-clearance-on-its-digital-pathology-solution-for-diagnostic-use.html
                
               (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR74" id="ref-link-section-d5807050e1145">74</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 122" title="Indica Labs. Indica Labs Receives First FDA Clearance for HALO AP Dx Digital Pathology Platform. Indica Labs 
                indicalab.com/news/indica-labs-receives-first-fda-clearance-for-halo-ap-dx-digital-pathology-platform/
                
               (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR122" id="ref-link-section-d5807050e1148">122</a></sup>, the integration of AI technologies into clinical workflows remains in the early stages<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 97" title="Eloy, C. et al. Artificial intelligence-assisted cancer diagnosis improves the efficiency of pathologists in prostatic biopsies. Virchows Arch. 482, 595–604 (2023)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR97" id="ref-link-section-d5807050e1152">97</a></sup>. This discrepancy often stems from an academic emphasis on developing advanced algorithms that, although innovative, can sometimes overshadow the equally critical focus on rigorous validation and meeting the bar for regulatory approval, key for clinical adoption<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 116" title="Zarella, M. D. et al. Artificial intelligence and digital pathology: clinical promise and deployment considerations. J. Med. Imaging 10, 051802 (2023)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR116" id="ref-link-section-d5807050e1156">116</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Hosseini, M. S. et al. Computational pathology: a survey review and the way forward. J. Pathol. Inf. 15, 100357 (2024)." href="https://www.nature.com/articles/#ref-CR123" id="ref-link-section-d5807050e1159">123</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Ghassemi, M., Oakden-Rayner, L. &amp; Beam, A. L. The false hope of current approaches to explainable artificial intelligence in health care. Lancet Digital Health 3, e745–e750 (2021)." href="https://www.nature.com/articles/#ref-CR124" id="ref-link-section-d5807050e1159_1">124</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 125" title="Chauhan, C. &amp; Gullapalli, R. R. Ethics of AI in pathology: current paradigms and emerging issues. Am. J. Pathol. 191, 1673–1683 (2021)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR125" id="ref-link-section-d5807050e1162">125</a></sup>. Certain countries are addressing this issue with stringent regulation aimed at balancing financial incentives to adopt new technologies while ensuring patient safety. For example, the Chinese State Council released the New Generation Artificial Intelligence Development Plan in 2017 that lists strategies and regulatory milestones to make China the global leader in AI by 2030 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 126" title="Roberts, H. et al. in Ethics, Governance, and Policies in Artificial Intelligence (ed. Floridi, L.) 47–79 (Springer, 2021)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR126" id="ref-link-section-d5807050e1166">126</a></sup>). Regulatory bodies oversee the process and incentivize local projects, particularly technology start-ups through government subsidies<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 126" title="Roberts, H. et al. in Ethics, Governance, and Policies in Artificial Intelligence (ed. Floridi, L.) 47–79 (Springer, 2021)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR126" id="ref-link-section-d5807050e1170">126</a></sup>. In Japan, no binding regulations constrain the use of AI<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 127" title="Habuka, H. Japan’s Approach to AI Regulation and its Impact on the 2023 G7 Presidency (Center for Strategic &amp; International Studies, 2023)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR127" id="ref-link-section-d5807050e1175">127</a></sup>; however, the government published the Social Principles of Human-Centric AI in 2019 to set forth principles focusing on human dignity, diversity, inclusion and sustainability for implementing AI in society<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 127" title="Habuka, H. Japan’s Approach to AI Regulation and its Impact on the 2023 G7 Presidency (Center for Strategic &amp; International Studies, 2023)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR127" id="ref-link-section-d5807050e1179">127</a></sup>. This effort aims to facilitate wider adoption and to provide a model for institutions other than that developing a specific AI-based model to use such capabilities while ensuring safety and consistency across practices.

Thorough validation is another key requirement for safe and effective deployment of AI tools in clinical settings<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 116" title="Zarella, M. D. et al. Artificial intelligence and digital pathology: clinical promise and deployment considerations. J. Med. Imaging 10, 051802 (2023)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR116" id="ref-link-section-d5807050e1186">116</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Hosseini, M. S. et al. Computational pathology: a survey review and the way forward. J. Pathol. Inf. 15, 100357 (2024)." href="https://www.nature.com/articles/#ref-CR123" id="ref-link-section-d5807050e1189">123</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Ghassemi, M., Oakden-Rayner, L. &amp; Beam, A. L. The false hope of current approaches to explainable artificial intelligence in health care. Lancet Digital Health 3, e745–e750 (2021)." href="https://www.nature.com/articles/#ref-CR124" id="ref-link-section-d5807050e1189_1">124</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 125" title="Chauhan, C. &amp; Gullapalli, R. R. Ethics of AI in pathology: current paradigms and emerging issues. Am. J. Pathol. 191, 1673–1683 (2021)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR125" id="ref-link-section-d5807050e1192">125</a></sup>. Of the 521 AI devices authorized by the FDA from 1995 to 2022, 56% underwent clinical validation (28% retrospectively and 28% prospectively); however, no clinical data were available for 43% of them<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 128" title="Chouffani El Fassi, S. et al. Not all AI health tools with regulatory authorization are clinically validated. Nat. Med. 30, 2718–2720 (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR128" id="ref-link-section-d5807050e1196">128</a></sup>. The majority of pathologists interviewed across the USA, Europe and Asia as part of our research emphasized the importance of extensive validation for prognostic and predictive biomarkers to mitigate potential risks to patients (A.A. and S. Bharadwaj, unpublished results). They further noted that AI tools must be validated in retrospective and prospective clinical trial datasets to establish solid evidence for their clinical viability. The inclusion of the ArteraAI Prostate Test<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 87" title="Artera. For Patients. Artera 
                https://artera.ai/for-patients
                
               (2025)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR87" id="ref-link-section-d5807050e1200">87</a></sup> in the NCCN guidelines<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 129" title="Businesswire. ArteraAI Announced as the First-and-Only Predictive Test for Therapy Personalization in the 2024 NCCN Guidelines® for Prostate Cancer. Businesswire 
                www.businesswire.com/news/home/20240304893588/en/ArteraAI-Announced-as-the-First-and-Only-Predictive-Test-for-Therapy-Personalization-in-the-2024-NCCN-Guidelines%C2%AE-for-Prostate-Cancer
                
               (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR129" id="ref-link-section-d5807050e1204">129</a></sup> is a key milestone for the use of AI-based tools in digital pathology<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 89" title="Schaeffer, E. M. et al. NCCN guidelines® insights: prostate cancer, version 3.2024. J. Natl Compr. Cancer Netw. 22, 140–150 (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR89" id="ref-link-section-d5807050e1208">89</a></sup>. Moreover, the supplementary assessment of Paige Prostate in three hospitals in the UK in the same year<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 130" title="Businesswire. Three UK-Based Hospital Systems Exploring the Power of Paige’s AI for Diagnosing Prostate Cancer in Live Clinical Settings. Businesswire 
                www.businesswire.com/news/home/20240722858117/en/Three-UK-Based-Hospital-Systems-Exploring-the-Power-of-Paige%E2%80%99s-AI-for-Diagnosing-Prostate-Cancer-in-Live-Clinical-Settings
                
               (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR130" id="ref-link-section-d5807050e1213">130</a></sup> provides an opportunity to assess the utility of this tool in cancer grading and detection. A detailed evaluation of the uptake of these tools in clinical settings over the next few years will provide key insights into their role in treatment decisions, the factors influencing regulatory considerations and financial implications (including implementation and maintenance costs), which all affect clinical adoption.

As AI technologies are being integrated into digital pathology, updating the skills of pathologists is increasingly important<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 104" title="Kiran, N. et al. Digital pathology: transforming diagnosis in the digital age. Cureus 15, e44620 (2023)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR104" id="ref-link-section-d5807050e1220">104</a></sup>. Training programmes should focus on the complementary role of AI in clinical workflows, helping pathologists to use these technologies to improve diagnostic precision and efficiency<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 104" title="Kiran, N. et al. Digital pathology: transforming diagnosis in the digital age. Cureus 15, e44620 (2023)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR104" id="ref-link-section-d5807050e1224">104</a></sup>. By assisting in identifying areas of interest in pathology images, AI enables pathologists to focus on detailed analysis, ultimately improving productivity and ensuring accurate evaluations<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 131" title="Kim, I., Kang, K., Song, Y. &amp; Kim, T.-J. Application of artificial intelligence in pathology: trends and challenges. Diagnostics 12, 2794 (2022)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR131" id="ref-link-section-d5807050e1228">131</a></sup>. The majority of pathologists we interviewed for our research noted that, in training environments, AI should be viewed as an augmentative tool that enhances, rather than replaces, the role of the pathologist.

Herein, we address key aspects related to AI adoption such as technological advances, regulatory trends, reimbursement frameworks, clinical implementation and commercial investment; however, we acknowledge that we have omitted several important factors that are beyond the scope of this Perspective. These include federal research initiatives and institutional funding crucial for shaping the research environment and accelerating AI development as well as the effect of AI on workforce-related aspects, such as the evolving roles of technicians in AI-based digital pathology workflows and the availability of training programmes to support AI adoption. The role of data privacy regulations, and ethical and legal guidelines are not discussed either, and will be essential for understanding the broader implications of AI adoption in digital pathology. These aforementioned limitations reflect the complex nature of AI regulation and adoption, warranting continued research.

## Conclusions

In 2019, we made key predictions about the future of AI applications for digital pathology in oncology, anticipating rapid technological advances, an increased focus on regulatory approvals and two important needs: robust validation datasets and structured reimbursement strategies<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Bera, K., Schalper, K. A., Rimm, D. L., Velcheti, V. &amp; Madabhushi, A. Artificial intelligence in digital pathology – new tools for diagnosis and precision oncology. Nat. Rev. Clin. Oncol. 16, 703–715 (2019)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR7" id="ref-link-section-d5807050e1244">7</a></sup>. After 5 years, some of these predictions have materialized — in particular, the integration of AI into clinical workflows<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="Shafi, S. &amp; Parwani, A. V. Artificial intelligence in diagnostic pathology. Diagn. Pathol. 18, 109 (2023)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR5" id="ref-link-section-d5807050e1248">5</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Baxi, V., Edwards, R., Montalto, M. &amp; Saha, S. Digital pathology and artificial intelligence in translational medicine and clinical practice. Mod. Pathol. 35, 23–32 (2022)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR6" id="ref-link-section-d5807050e1251">6</a></sup>, demonstrated by the FDA approval of Paige Prostate<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 97" title="Eloy, C. et al. Artificial intelligence-assisted cancer diagnosis improves the efficiency of pathologists in prostatic biopsies. Virchows Arch. 482, 595–604 (2023)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR97" id="ref-link-section-d5807050e1255">97</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 132" title="Kanan, C. et al. Independent validation of Paige Prostate: assessing clinical benefit of an artificial intelligence tool within a digital diagnostic pathology laboratory workflow [abstract]. J. Clin. Oncol. 38 (Suppl. 15), e14076 (2020)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR132" id="ref-link-section-d5807050e1258">132</a></sup>. However, the expected increase in regulatory approvals has not occurred at a rate comparable to that of other AI applications in other medical fields<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 66" title="Joshi, G. et al. FDA-approved artificial intelligence and machine learning (AI/ML)-enabled medical devices: an updated landscape. Electronics 13, 498 (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR66" id="ref-link-section-d5807050e1262">66</a></sup>.

The clinical integration of AI-based applications in routine pathology remains constrained by gaps in infrastructure<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 133" title="Zehra, T., Parwani, A., Abdul-Ghafar, J. &amp; Ahmad, Z. A suggested way forward for adoption of AI-enabled digital pathology in low resource organizations in the developing world. Diagn. Pathol. 18, 68 (2023)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR133" id="ref-link-section-d5807050e1269">133</a></sup>, reimbursement pathways<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 134" title="Schwen, L. O., Kiehl, T.-R., Carvalho, R., Zerbe, N. &amp; Homeyer, A. Digitization of pathology labs: a review of lessons learned. Lab. Invest. 103, 100244 (2023)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR134" id="ref-link-section-d5807050e1273">134</a></sup> and regulatory frameworks<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 133" title="Zehra, T., Parwani, A., Abdul-Ghafar, J. &amp; Ahmad, Z. A suggested way forward for adoption of AI-enabled digital pathology in low resource organizations in the developing world. Diagn. Pathol. 18, 68 (2023)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR133" id="ref-link-section-d5807050e1277">133</a></sup>. Addressing these barriers will require targeted efforts from academia, industry and regulatory bodies to refine standards, enhance interpretability and ensure scalability in clinical settings. The future of AI in digital pathology depends on collaborative efforts to establish evidence-based guidelines<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 135" title="Zhang, D. Y. et al. Implementation of digital pathology and artificial intelligence in routine pathology practice. Lab. Invest. 104, 102111 (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR135" id="ref-link-section-d5807050e1281">135</a></sup> and sustainable implementation models<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 133" title="Zehra, T., Parwani, A., Abdul-Ghafar, J. &amp; Ahmad, Z. A suggested way forward for adoption of AI-enabled digital pathology in low resource organizations in the developing world. Diagn. Pathol. 18, 68 (2023)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR133" id="ref-link-section-d5807050e1285">133</a></sup> to advance and integrate them into routine clinical practice.

## References

1. Rajpurkar, P., Chen, E., Banerjee, O. & Topol, E. J. AI in health and medicine. *Nat. Med.* **28**, 31–38 (2022).

[Article](https://doi.org/10.1038%2Fs41591-021-01614-0)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhslCntr4%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35058619)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=AI%20in%20health%20and%20medicine&journal=Nat.%20Med.&doi=10.1038%2Fs41591-021-01614-0&volume=28&pages=31-38&publication_year=2022&author=Rajpurkar%2CP&author=Chen%2CE&author=Banerjee%2CO&author=Topol%2CEJ)
2. Topol, E. J. High-performance medicine: the convergence of human and artificial intelligence. *Nat. Med.* **25**, 44–56 (2019).

[Article](https://doi.org/10.1038%2Fs41591-018-0300-7)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXmvVOgsbs%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30617339)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=High-performance%20medicine%3A%20the%20convergence%20of%20human%20and%20artificial%20intelligence&journal=Nat.%20Med.&doi=10.1038%2Fs41591-018-0300-7&volume=25&pages=44-56&publication_year=2019&author=Topol%2CEJ)
3. Aeffner, F. et al. Introduction to digital image analysis in whole-slide imaging: a white paper from the Digital Pathology Association. *J. Pathol. Inform.* **10**, 9 (2019).

[Article](https://doi.org/10.4103%2Fjpi.jpi_82_18)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30984469)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6437786)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Introduction%20to%20digital%20image%20analysis%20in%20whole-slide%20imaging%3A%20a%20white%20paper%20from%20the%20Digital%20Pathology%20Association&journal=J.%20Pathol.%20Inform.&doi=10.4103%2Fjpi.jpi_82_18&volume=10&publication_year=2019&author=Aeffner%2CF)
4. van der Laak, J., Litjens, G. & Ciompi, F. Deep learning in histopathology: the path to the clinic. *Nat. Med.* **27**, 775–784 (2021).

[Article](https://doi.org/10.1038%2Fs41591-021-01343-4)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33990804)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Deep%20learning%20in%20histopathology%3A%20the%20path%20to%20the%20clinic&journal=Nat.%20Med.&doi=10.1038%2Fs41591-021-01343-4&volume=27&pages=775-784&publication_year=2021&author=Laak%2CJ&author=Litjens%2CG&author=Ciompi%2CF)
5. Shafi, S. & Parwani, A. V. Artificial intelligence in diagnostic pathology. *Diagn. Pathol.* **18**, 109 (2023).

[Article](https://link.springer.com/doi/10.1186/s13000-023-01375-z)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37784122)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10546747)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Artificial%20intelligence%20in%20diagnostic%20pathology&journal=Diagn.%20Pathol.&doi=10.1186%2Fs13000-023-01375-z&volume=18&publication_year=2023&author=Shafi%2CS&author=Parwani%2CAV)
6. Baxi, V., Edwards, R., Montalto, M. & Saha, S. Digital pathology and artificial intelligence in translational medicine and clinical practice. *Mod. Pathol.* **35**, 23–32 (2022).

[Article](https://doi.org/10.1038%2Fs41379-021-00919-2)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XktVCmtr0%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34611303)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Digital%20pathology%20and%20artificial%20intelligence%20in%20translational%20medicine%20and%20clinical%20practice&journal=Mod.%20Pathol.&doi=10.1038%2Fs41379-021-00919-2&volume=35&pages=23-32&publication_year=2022&author=Baxi%2CV&author=Edwards%2CR&author=Montalto%2CM&author=Saha%2CS)
7. Bera, K., Schalper, K. A., Rimm, D. L., Velcheti, V. & Madabhushi, A. Artificial intelligence in digital pathology – new tools for diagnosis and precision oncology. *Nat. Rev. Clin. Oncol.* **16**, 703–715 (2019).

[Article](https://doi.org/10.1038%2Fs41571-019-0252-y)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31399699)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6880861)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Artificial%20intelligence%20in%20digital%20pathology%20%E2%80%93%20new%20tools%20for%20diagnosis%20and%20precision%20oncology&journal=Nat.%20Rev.%20Clin.%20Oncol.&doi=10.1038%2Fs41571-019-0252-y&volume=16&pages=703-715&publication_year=2019&author=Bera%2CK&author=Schalper%2CKA&author=Rimm%2CDL&author=Velcheti%2CV&author=Madabhushi%2CA)
8. O’Shea, K. & Nash, R. An introduction to convolutional neural networks. Preprint at *arXiv* [https://doi.org/10.48550/arXiv.1511.08458](https://doi.org/10.48550/arXiv.1511.08458) (2015).
9. Coudray, N. et al. Classification and mutation prediction from non–small cell lung cancer histopathology images using deep learning. *Nat. Med.* **24**, 1559–1567 (2018).

[Article](https://doi.org/10.1038%2Fs41591-018-0177-5)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhslCjtrbJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30224757)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9847512)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Classification%20and%20mutation%20prediction%20from%20non%E2%80%93small%20cell%20lung%20cancer%20histopathology%20images%20using%20deep%20learning&journal=Nat.%20Med.&doi=10.1038%2Fs41591-018-0177-5&volume=24&pages=1559-1567&publication_year=2018&author=Coudray%2CN)
10. Dosovitskiy, A. et al. An image is worth 16x16 words: transformers for image recognition at scale. Preprint at *arXiv* [https://doi.org/10.48550/arXiv.2010.11929](https://doi.org/10.48550/arXiv.2010.11929) (2020).
11. Chen, R. J. et al. Towards a general-purpose foundation model for computational pathology. *Nat. Med.* **30**, 850–862 (2024).

[Article](https://doi.org/10.1038%2Fs41591-024-02857-3)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXmtlyns7s%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38504018)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11403354)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Towards%20a%20general-purpose%20foundation%20model%20for%20computational%20pathology&journal=Nat.%20Med.&doi=10.1038%2Fs41591-024-02857-3&volume=30&pages=850-862&publication_year=2024&author=Chen%2CRJ)
12. Xu, H. et al. A whole-slide foundation model for digital pathology from real-world data. *Nature* **630**, 181–188 (2024).

[Article](https://doi.org/10.1038%2Fs41586-024-07441-w)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXhtFeksbvE)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38778098)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11153137)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20whole-slide%20foundation%20model%20for%20digital%20pathology%20from%20real-world%20data&journal=Nature&doi=10.1038%2Fs41586-024-07441-w&volume=630&pages=181-188&publication_year=2024&author=Xu%2CH)
13. Vorontsov, E. A foundation model for clinical-grade computational pathology and rare cancers detection. *Nat. Med.* **30**, 2924–2935 (2024).

[Article](https://doi.org/10.1038%2Fs41591-024-03141-0)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXhs1ajt77E)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=39039250)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11485232)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20foundation%20model%20for%20clinical-grade%20computational%20pathology%20and%20rare%20cancers%20detection&journal=Nat.%20Med.&doi=10.1038%2Fs41591-024-03141-0&volume=30&pages=2924-2935&publication_year=2024&author=Vorontsov%2CE)
14. Dippel, J. et al. RudolfV: a foundation model by pathologists for pathologists. Preprint at *arXiv* [https://doi.org/10.48550/arXiv.2401.04079](https://doi.org/10.48550/arXiv.2401.04079) (2024).
15. Wang, X. et al. A pathology foundation model for cancer diagnosis and prognosis prediction. *Nature* **634**, 970–978 (2024).

[Article](https://doi.org/10.1038%2Fs41586-024-07894-z)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXhvFSqurzL)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=39232164)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20pathology%20foundation%20model%20for%20cancer%20diagnosis%20and%20prognosis%20prediction&journal=Nature&doi=10.1038%2Fs41586-024-07894-z&volume=634&pages=970-978&publication_year=2024&author=Wang%2CX)
16. Ma, J. et al. Towards a generalizable pathology foundation model via unified knowledge distillation. Preprint at *arXiv* [https://doi.org/10.48550/arXiv.2407.18449](https://doi.org/10.48550/arXiv.2407.18449) (2024).
17. Yang, Z. et al. A foundation model for generalizable cancer diagnosis and survival prediction from histopathological images. Preprint at *bioRxiv* [https://doi.org/10.1101/2024.05.16.594499](https://doi.org/10.1101/2024.05.16.594499) (2024).
18. Hua, S., Yan, F., Shen, T., Ma, L. & Zhang, X. PathoDuet: foundation models for pathological slide analysis of H&E and IHC stains. *Med. Image Anal.* **97**, 103289 (2024).

[Article](https://doi.org/10.1016%2Fj.media.2024.103289)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=39106763)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=PathoDuet%3A%20foundation%20models%20for%20pathological%20slide%20analysis%20of%20H%26E%20and%20IHC%20stains&journal=Med.%20Image%20Anal.&doi=10.1016%2Fj.media.2024.103289&volume=97&publication_year=2024&author=Hua%2CS&author=Yan%2CF&author=Shen%2CT&author=Ma%2CL&author=Zhang%2CX)
19. Nechaev, D., Pchelnikov, A. & Ivanova, E. Hibou: a family of foundational vision transformers for pathology. Preprint at *arXiv* [https://doi.org/10.48550/arXiv.2406.05074](https://doi.org/10.48550/arXiv.2406.05074) (2024).
20. Lu, M. Y. et al. A visual-language foundation model for computational pathology. *Nat. Med.* **30**, 863–874 (2024).

[Article](https://doi.org/10.1038%2Fs41591-024-02856-4)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXmtlyns7o%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38504017)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11384335)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20visual-language%20foundation%20model%20for%20computational%20pathology&journal=Nat.%20Med.&doi=10.1038%2Fs41591-024-02856-4&volume=30&pages=863-874&publication_year=2024&author=Lu%2CMY)
21. Zimmermann, E. et al. Virchow2: scaling self-supervised mixed magnification models in pathology. Preprint at *arXiv* [https://doi.org/10.48550/arXiv.2408.00738](https://doi.org/10.48550/arXiv.2408.00738) (2024).
22. Juyal, D. et al. PLUTO: pathology-universal transformer. Preprint at *arXiv* [https://doi.org/10.48550/arXiv.2405.07905](https://doi.org/10.48550/arXiv.2405.07905) (2024).
23. Li, H. et al. A prognostic and predictive computational pathology immune signature for ductal carcinoma in situ: retrospective results from a cohort within the UK/ANZ DCIS trial. *Lancet Digital Health* **6**, e562–e569 (2024).

[Article](https://doi.org/10.1016%2FS2589-7500%2824%2900116-X)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXhsFarsrnF)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38987116)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20prognostic%20and%20predictive%20computational%20pathology%20immune%20signature%20for%20ductal%20carcinoma%20in%20situ%3A%20retrospective%20results%20from%20a%20cohort%20within%20the%20UK%2FANZ%20DCIS%20trial&journal=Lancet%20Digital%20Health&doi=10.1016%2FS2589-7500%2824%2900116-X&volume=6&pages=e562-e569&publication_year=2024&author=Li%2CH)
24. Lu, M. Y. et al. AI-based pathology predicts origins for cancers of unknown primary. *Nature* **594**, 106–110 (2021).

[Article](https://doi.org/10.1038%2Fs41586-021-03512-4)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtVCrsb3M)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33953404)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=AI-based%20pathology%20predicts%20origins%20for%20cancers%20of%20unknown%20primary&journal=Nature&doi=10.1038%2Fs41586-021-03512-4&volume=594&pages=106-110&publication_year=2021&author=Lu%2CMY)
25. Bulten, W. et al. Artificial intelligence for diagnosis and Gleason grading of prostate cancer: the PANDA challenge. *Nat. Med.* **28**, 154–163 (2022).

[Article](https://doi.org/10.1038%2Fs41591-021-01620-2)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xht1Kqsb4%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35027755)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8799467)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Artificial%20intelligence%20for%20diagnosis%20and%20Gleason%20grading%20of%20prostate%20cancer%3A%20the%20PANDA%20challenge&journal=Nat.%20Med.&doi=10.1038%2Fs41591-021-01620-2&volume=28&pages=154-163&publication_year=2022&author=Bulten%2CW)
26. Bulten, W. et al. Automated deep-learning system for Gleason grading of prostate cancer using biopsies: a diagnostic study. *Lancet Oncol.* **21**, 233–241 (2020).

[Article](https://doi.org/10.1016%2FS1470-2045%2819%2930739-9)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31926805)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Automated%20deep-learning%20system%20for%20Gleason%20grading%20of%20prostate%20cancer%20using%20biopsies%3A%20a%20diagnostic%20study&journal=Lancet%20Oncol.&doi=10.1016%2FS1470-2045%2819%2930739-9&volume=21&pages=233-241&publication_year=2020&author=Bulten%2CW)
27. Aggarwal, A. et al. Computational pathology identifies immune-mediated collagen disruption to predict clinical outcomes in gynecologic malignancies. *Commun. Med.* **4**, 2 (2024).

[Article](https://doi.org/10.1038%2Fs43856-023-00428-0)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXmslaisg%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38172536)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10764846)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Computational%20pathology%20identifies%20immune-mediated%20collagen%20disruption%20to%20predict%20clinical%20outcomes%20in%20gynecologic%20malignancies&journal=Commun.%20Med.&doi=10.1038%2Fs43856-023-00428-0&volume=4&publication_year=2024&author=Aggarwal%2CA)
28. Li, H. et al. Collagen fiber orientation disorder from H&E images is prognostic for early stage breast cancer: clinical trial validation. *NPJ Breast Cancer* **7**, 104 (2021).

[Article](https://doi.org/10.1038%2Fs41523-021-00310-z)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhvFGntLnP)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34362928)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8346522)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Collagen%20fiber%20orientation%20disorder%20from%20H%26E%20images%20is%20prognostic%20for%20early%20stage%20breast%20cancer%3A%20clinical%20trial%20validation&journal=NPJ%20Breast%20Cancer&doi=10.1038%2Fs41523-021-00310-z&volume=7&publication_year=2021&author=Li%2CH)
29. Bilal, M. et al. Development and validation of a weakly supervised deep learning framework to predict the status of molecular pathways and key mutations in colorectal cancer from routine histology images: a retrospective study. *Lancet Digital Health* **3**, e763–e772 (2021).

[Article](https://doi.org/10.1016%2FS2589-7500%2821%2900180-1)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXis1Skt7o%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34686474)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Development%20and%20validation%20of%20a%20weakly%20supervised%20deep%20learning%20framework%20to%20predict%20the%20status%20of%20molecular%20pathways%20and%20key%20mutations%20in%20colorectal%20cancer%20from%20routine%20histology%20images%3A%20a%20retrospective%20study&journal=Lancet%20Digital%20Health&doi=10.1016%2FS2589-7500%2821%2900180-1&volume=3&pages=e763-e772&publication_year=2021&author=Bilal%2CM)
30. Lugli, A., Zlobec, I., Berger, M. D., Kirsch, R. & Nagtegaal, I. D. Tumour budding in solid cancers. *Nat. Rev. Clin. Oncol.* **18**, 101–115 (2021).

[Article](https://doi.org/10.1038%2Fs41571-020-0422-y)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32901132)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Tumour%20budding%20in%20solid%20cancers&journal=Nat.%20Rev.%20Clin.%20Oncol.&doi=10.1038%2Fs41571-020-0422-y&volume=18&pages=101-115&publication_year=2021&author=Lugli%2CA&author=Zlobec%2CI&author=Berger%2CMD&author=Kirsch%2CR&author=Nagtegaal%2CID)
31. Neppl, C., Zlobec, I., Schmid, R. A. & Berezowska, S. Validation of the International Tumor Budding Consensus Conference (ITBCC) 2016 recommendation in squamous cell carcinoma of the lung – a single-center analysis of 354 cases. *Mod. Pathol.* **33**, 802–811 (2020).

[Article](https://doi.org/10.1038%2Fs41379-019-0413-7)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXitlajs7zN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31796876)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Validation%20of%20the%20International%20Tumor%20Budding%20Consensus%20Conference%20%28ITBCC%29%202016%20recommendation%20in%20squamous%20cell%20carcinoma%20of%20the%20lung%20%E2%80%93%20a%20single-center%20analysis%20of%20354%20cases&journal=Mod.%20Pathol.&doi=10.1038%2Fs41379-019-0413-7&volume=33&pages=802-811&publication_year=2020&author=Neppl%2CC&author=Zlobec%2CI&author=Schmid%2CRA&author=Berezowska%2CS)
32. Dawson, H. et al. Tumour budding/T cell infiltrates in colorectal cancer: proposal of a novel combined score. *Histopathology* **76**, 572–580 (2020).

[Article](https://doi.org/10.1111%2Fhis.14006)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31560788)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Tumour%20budding%2FT%20cell%20infiltrates%20in%20colorectal%20cancer%3A%20proposal%20of%20a%20novel%20combined%20score&journal=Histopathology&doi=10.1111%2Fhis.14006&volume=76&pages=572-580&publication_year=2020&author=Dawson%2CH)
33. Xie, W. et al. Prostate cancer risk stratification via nondestructive 3D pathology with deep learning-assisted gland analysis. *Cancer Res.* **82**, 334–345 (2022).

[Article](https://doi.org/10.1158%2F0008-5472.CAN-21-2843)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xmt1aht7s%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34853071)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Prostate%20cancer%20risk%20stratification%20via%20nondestructive%203D%20pathology%20with%20deep%20learning-assisted%20gland%20analysis&journal=Cancer%20Res.&doi=10.1158%2F0008-5472.CAN-21-2843&volume=82&pages=334-345&publication_year=2022&author=Xie%2CW)
34. Lu, C. et al. A prognostic model for overall survival of patients with early-stage non-small cell lung cancer: a multicentre, retrospective study. *Lancet Digital Health* **2**, e594–e606 (2020).

[Article](https://doi.org/10.1016%2FS2589-7500%2820%2930225-9)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33163952)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20prognostic%20model%20for%20overall%20survival%20of%20patients%20with%20early-stage%20non-small%20cell%20lung%20cancer%3A%20a%20multicentre%2C%20retrospective%20study&journal=Lancet%20Digital%20Health&doi=10.1016%2FS2589-7500%2820%2930225-9&volume=2&pages=e594-e606&publication_year=2020&author=Lu%2CC)
35. Lu, C. et al. Feature-driven local cell graph (FLocK): new computational pathology-based descriptors for prognosis of lung cancer and HPV status of oropharyngeal cancers. *Med. Image Anal.* **68**, 101903 (2021).

[Article](https://doi.org/10.1016%2Fj.media.2020.101903)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33352373)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Feature-driven%20local%20cell%20graph%20%28FLocK%29%3A%20new%20computational%20pathology-based%20descriptors%20for%20prognosis%20of%20lung%20cancer%20and%20HPV%20status%20of%20oropharyngeal%20cancers&journal=Med.%20Image%20Anal.&doi=10.1016%2Fj.media.2020.101903&volume=68&publication_year=2021&author=Lu%2CC)
36. Fashi, P. A., Hemati, S., Babaie, M., Gonzalez, R. & Tizhoosh, H. R. A self-supervised contrastive learning approach for whole slide image representation in digital pathology. *J. Pathol. Inf.* **13**, 100133 (2022).

[Article](https://doi.org/10.1016%2Fj.jpi.2022.100133)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20self-supervised%20contrastive%20learning%20approach%20for%20whole%20slide%20image%20representation%20in%20digital%20pathology&journal=J.%20Pathol.%20Inf.&doi=10.1016%2Fj.jpi.2022.100133&volume=13&publication_year=2022&author=Fashi%2CPA&author=Hemati%2CS&author=Babaie%2CM&author=Gonzalez%2CR&author=Tizhoosh%2CHR)
37. Tsai, P.-C. et al. Histopathology images predict multi-omics aberrations and prognoses in colorectal cancer patients. *Nat. Commun.* **14**, 2102 (2023).

[Article](https://doi.org/10.1038%2Fs41467-023-37179-4)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXot1Wjsr8%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37055393)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10102208)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Histopathology%20images%20predict%20multi-omics%20aberrations%20and%20prognoses%20in%20colorectal%20cancer%20patients&journal=Nat.%20Commun.&doi=10.1038%2Fs41467-023-37179-4&volume=14&publication_year=2023&author=Tsai%2CP-C)
38. Marostica, E. et al. Development of a histopathology informatics pipeline for classification and prediction of clinical outcomes in subtypes of renal cell carcinoma. *Clin. Cancer Res.* **27**, 2868–2878 (2021).

[Article](https://doi.org/10.1158%2F1078-0432.CCR-20-4119)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtl2rsL3I)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33722896)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Development%20of%20a%20histopathology%20informatics%20pipeline%20for%20classification%20and%20prediction%20of%20clinical%20outcomes%20in%20subtypes%20of%20renal%20cell%20carcinoma&journal=Clin.%20Cancer%20Res.&doi=10.1158%2F1078-0432.CCR-20-4119&volume=27&pages=2868-2878&publication_year=2021&author=Marostica%2CE)
39. Campanella, G., Vanderbilt, C. & Fuchs, T. Computational pathology at health system scale – self-supervised foundation models from three billion images. Preprint at *arXiv* [https://doi.org/10.48550/arXiv.2310.07033](https://doi.org/10.48550/arXiv.2310.07033) (2024).
40. Da Silva, L. M. et al. Independent real‐world application of a clinical‐grade automated prostate cancer detection system. *J. Pathol.* **254**, 147–158 (2021).

[Article](https://doi.org/10.1002%2Fpath.5662)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33904171)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8252036)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Independent%20real%E2%80%90world%20application%20of%20a%20clinical%E2%80%90grade%20automated%20prostate%20cancer%20detection%20system&journal=J.%20Pathol.&doi=10.1002%2Fpath.5662&volume=254&pages=147-158&publication_year=2021&author=Silva%2CLM)
41. Corredor, G. et al. Spatial architecture and arrangement of tumor-infiltrating lymphocytes for predicting likelihood of recurrence in early-stage non-small cell lung cancer. *Clin. Cancer Res.* **25**, 1526–1534 (2019).

[Article](https://doi.org/10.1158%2F1078-0432.CCR-18-2013)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtlOmt7vK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30201760)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Spatial%20architecture%20and%20arrangement%20of%20tumor-infiltrating%20lymphocytes%20for%20predicting%20likelihood%20of%20recurrence%20in%20early-stage%20non-small%20cell%20lung%20cancer&journal=Clin.%20Cancer%20Res.&doi=10.1158%2F1078-0432.CCR-18-2013&volume=25&pages=1526-1534&publication_year=2019&author=Corredor%2CG)
42. Koyuncu, C. F. et al. Computerized tumor multinucleation index (MuNI) is prognostic in p16<sup>+</sup> oropharyngeal carcinoma. *J. Clin. Invest.* **131**, e145488 (2021).

[Article](https://doi.org/10.1172%2FJCI145488)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33651718)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8075166)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Computerized%20tumor%20multinucleation%20index%20%28MuNI%29%20is%20prognostic%20in%20p16%2B%20oropharyngeal%20carcinoma&journal=J.%20Clin.%20Invest.&doi=10.1172%2FJCI145488&volume=131&publication_year=2021&author=Koyuncu%2CCF)
43. Balkenhol, M. C. A. et al. Optimized tumour infiltrating lymphocyte assessment for triple negative breast cancer prognostics. *Breast* **56**, 78–87 (2021).

[Article](https://doi.org/10.1016%2Fj.breast.2021.02.007)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33640523)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7933536)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Optimized%20tumour%20infiltrating%20lymphocyte%20assessment%20for%20triple%20negative%20breast%20cancer%20prognostics&journal=Breast&doi=10.1016%2Fj.breast.2021.02.007&volume=56&pages=78-87&publication_year=2021&author=Balkenhol%2CMCA)
44. Pinckaers, H. et al. Predicting biochemical recurrence of prostate cancer with artificial intelligence. *Commun. Med.* **2**, 64 (2022).

[Article](https://doi.org/10.1038%2Fs43856-022-00126-3)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35693032)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9177591)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Predicting%20biochemical%20recurrence%20of%20prostate%20cancer%20with%20artificial%20intelligence&journal=Commun.%20Med.&doi=10.1038%2Fs43856-022-00126-3&volume=2&publication_year=2022&author=Pinckaers%2CH)
45. Vendittelli, P. et al. Automatic quantification of tumor–stroma ratio as a prognostic marker for pancreatic cancer. *PLoS ONE* **19**, e0301969 (2024).

[Article](https://doi.org/10.1371%2Fjournal.pone.0301969)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXht1altbbO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38771787)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11108171)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Automatic%20quantification%20of%20tumor%E2%80%93stroma%20ratio%20as%20a%20prognostic%20marker%20for%20pancreatic%20cancer&journal=PLoS%20ONE&doi=10.1371%2Fjournal.pone.0301969&volume=19&publication_year=2024&author=Vendittelli%2CP)
46. Aggarwal, A. et al. Image analysis uncovers associations between immune landscape, collagen structure, and neoadjuvant chemotherapy in high-grade serous ovarian carcinomas. *Heliyon* **10**, e33618 (2024).

[Article](https://doi.org/10.1016%2Fj.heliyon.2024.e33618)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXhsVWns7vJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=39035539)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11259872)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Image%20analysis%20uncovers%20associations%20between%20immune%20landscape%2C%20collagen%20structure%2C%20and%20neoadjuvant%20chemotherapy%20in%20high-grade%20serous%20ovarian%20carcinomas&journal=Heliyon&doi=10.1016%2Fj.heliyon.2024.e33618&volume=10&publication_year=2024&author=Aggarwal%2CA)
47. Li, H. et al. Quantitative nuclear histomorphometric features are predictive of Oncotype DX risk categories in ductal carcinoma in situ: preliminary findings. *Breast Cancer Res.* **21**, 114 (2019).

[Article](https://link.springer.com/doi/10.1186/s13058-019-1200-6)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31623652)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6798488)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Quantitative%20nuclear%20histomorphometric%20features%20are%20predictive%20of%20Oncotype%20DX%20risk%20categories%20in%20ductal%20carcinoma%20in%20situ%3A%20preliminary%20findings&journal=Breast%20Cancer%20Res.&doi=10.1186%2Fs13058-019-1200-6&volume=21&publication_year=2019&author=Li%2CH)
48. Geaney, A. et al. Translation of tissue-based artificial intelligence into clinical practice: from discovery to adoption. *Oncogene* **42**, 3545–3555 (2023).

[Article](https://doi.org/10.1038%2Fs41388-023-02857-6)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXitF2ksL3P)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37875656)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10673711)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Translation%20of%20tissue-based%20artificial%20intelligence%20into%20clinical%20practice%3A%20from%20discovery%20to%20adoption&journal=Oncogene&doi=10.1038%2Fs41388-023-02857-6&volume=42&pages=3545-3555&publication_year=2023&author=Geaney%2CA)
49. Balestriero, R. et al. A cookbook of self-supervised learning. Preprint at *arXiv* [https://doi.org/10.48550/arXiv.2304.12210](https://doi.org/10.48550/arXiv.2304.12210) (2023).
50. Unger, M. & Kather, J. N. A systematic analysis of deep learning in genomics and histopathology for precision oncology. *BMC Med. Genomics* **17**, 48 (2024).

[Article](https://link.springer.com/doi/10.1186/s12920-024-01796-9)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38317154)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10845449)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20systematic%20analysis%20of%20deep%20learning%20in%20genomics%20and%20histopathology%20for%20precision%20oncology&journal=BMC%20Med.%20Genomics&doi=10.1186%2Fs12920-024-01796-9&volume=17&publication_year=2024&author=Unger%2CM&author=Kather%2CJN)
51. Monabbati, S. et al. Machine vision-detected peritumoral lymphocytic aggregates are associated with disease-free survival in papillary thyroid carcinoma patients. *Lab. Invest.* **104**, 102168 (2024).

[Article](https://doi.org/10.1016%2Fj.labinv.2024.102168)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=39505213)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Machine%20vision-detected%20peritumoral%20lymphocytic%20aggregates%20are%20associated%20with%20disease-free%20survival%20in%20papillary%20thyroid%20carcinoma%20patients&journal=Lab.%20Invest.&doi=10.1016%2Fj.labinv.2024.102168&volume=104&publication_year=2024&author=Monabbati%2CS)
52. Monabbati, S. et al. A novel computational pathology approach for identifying gene signatures prognostic of disease-free survival for papillary thyroid carcinomas. *Eur. J. Cancer* **212**, 114326 (2024).

[Article](https://doi.org/10.1016%2Fj.ejca.2024.114326)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXitVeqs73J)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=39307037)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20novel%20computational%20pathology%20approach%20for%20identifying%20gene%20signatures%20prognostic%20of%20disease-free%20survival%20for%20papillary%20thyroid%20carcinomas&journal=Eur.%20J.%20Cancer&doi=10.1016%2Fj.ejca.2024.114326&volume=212&publication_year=2024&author=Monabbati%2CS)
53. Wang, X. et al. Spatial interplay patterns of cancer nuclei and tumor-infiltrating lymphocytes (TILs) predict clinical benefit for immune checkpoint inhibitors. *Sci. Adv.* **8**, eabn3966 (2022).

[Article](https://doi.org/10.1126%2Fsciadv.abn3966)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhvVSrtb%2FN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35648850)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9159577)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Spatial%20interplay%20patterns%20of%20cancer%20nuclei%20and%20tumor-infiltrating%20lymphocytes%20%28TILs%29%20predict%20clinical%20benefit%20for%20immune%20checkpoint%20inhibitors&journal=Sci.%20Adv.&doi=10.1126%2Fsciadv.abn3966&volume=8&publication_year=2022&author=Wang%2CX)
54. Arabyarmohammadi, S. et al. in *Medical Image Computing and Computer Assisted Intervention – MICCAI 2023* Part VI (eds Greenspan, H. et al.) 797–807 (Springer, 2023).
55. Monabbati, S., Khalighi, S., Fu, P., Asa, S. & Madabhushi, A. in *Medical Imaging 2023: Digital and Computational Pathology* Vol. 12471 (eds Tomaszewski, J. E. & Ward, A. D.) 284–293 (SPIE, 2023).
56. Nag, R. et al. Association of artificial intelligence-derived collagen disorder architecture (CoDA) features with survival outcome and objective response to immune checkpoint inhibitors in patients with head and neck squamous cell carcinoma \[abstract\]. *J. Clin. Oncol.* **42** (Suppl. 16), 6061 (2024).

[Article](https://doi.org/10.1200%2FJCO.2024.42.16_suppl.6061)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Association%20of%20artificial%20intelligence-derived%20collagen%20disorder%20architecture%20%28CoDA%29%20features%20with%20survival%20outcome%20and%20objective%20response%20to%20immune%20checkpoint%20inhibitors%20in%20patients%20with%20head%20and%20neck%20squamous%20cell%20carcinoma%20%5Babstract%5D&journal=J.%20Clin.%20Oncol.&doi=10.1200%2FJCO.2024.42.16_suppl.6061&volume=42&issue=Suppl.%2016&publication_year=2024&author=Nag%2CR)
57. Monabbati, S. et al. Automated analysis of computerized morphological features of cell clusters associated with malignancy on bile duct brushing whole slide images. *Cancer Med.* **12**, 6365–6378 (2023).

[Article](https://doi.org/10.1002%2Fcam4.5365)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36281473)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Automated%20analysis%20of%20computerized%20morphological%20features%20of%20cell%20clusters%20associated%20with%20malignancy%20on%20bile%20duct%20brushing%20whole%20slide%20images&journal=Cancer%20Med.&doi=10.1002%2Fcam4.5365&volume=12&pages=6365-6378&publication_year=2023&author=Monabbati%2CS)
58. Lu, M. Y. et al. Data-efficient and weakly supervised computational pathology on whole-slide images. *Nat. Biomed. Eng.* **5**, 555–570 (2021).

[Article](https://doi.org/10.1038%2Fs41551-020-00682-w)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33649564)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8711640)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Data-efficient%20and%20weakly%20supervised%20computational%20pathology%20on%20whole-slide%20images&journal=Nat.%20Biomed.%20Eng.&doi=10.1038%2Fs41551-020-00682-w&volume=5&pages=555-570&publication_year=2021&author=Lu%2CMY)
59. Oquab, M. et al. DINOv2: learning robust visual features without supervision. Preprint at *arXiv* [https://doi.org/10.48550/arXiv.2304.07193](https://doi.org/10.48550/arXiv.2304.07193) (2024).
60. Alfasly, S. et al. Foundation models for histopathology – fanfare or flair. *Mayo Clin. Proceedings: Digital Health* **2**, 165–174 (2024).

[Google Scholar](http://scholar.google.com/scholar_lookup?&title=Foundation%20models%20for%20histopathology%20%E2%80%93%20fanfare%20or%20flair&journal=Mayo%20Clin.%20Proceedings%3A%20Digital%20Health&volume=2&pages=165-174&publication_year=2024&author=Alfasly%2CS)
61. Spratt, D. E. et al. Artificial intelligence predictive model for hormone therapy use in prostate cancer. *NEJM Evid.* **2**, EVIDoa2300023 (2023).

[Article](https://doi.org/10.1056%2FEVIDoa2300023)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38320143)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11195914)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Artificial%20intelligence%20predictive%20model%20for%20hormone%20therapy%20use%20in%20prostate%20cancer&journal=NEJM%20Evid.&doi=10.1056%2FEVIDoa2300023&volume=2&publication_year=2023&author=Spratt%2CDE)
62. Boehm, K. M. et al. Multimodal data integration using machine learning improves risk stratification of high-grade serous ovarian cancer. *Nat. Cancer* **3**, 723–733 (2022).

[Article](https://doi.org/10.1038%2Fs43018-022-00388-9)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhvFGrurzJ)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35764743)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9239907)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Multimodal%20data%20integration%20using%20machine%20learning%20improves%20risk%20stratification%20of%20high-grade%20serous%20ovarian%20cancer&journal=Nat.%20Cancer&doi=10.1038%2Fs43018-022-00388-9&volume=3&pages=723-733&publication_year=2022&author=Boehm%2CKM)
63. Lu, C., Shiradkar, R. & Liu, Z. Integrating pathomics with radiomics and genomics for cancer prognosis: a brief review. *Chin. J. Cancer Res.* **33**, 563–573 (2021).

[Article](https://doi.org/10.21147%2Fj.issn.1000-9604.2021.05.03)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34815630)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8580801)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Integrating%20pathomics%20with%20radiomics%20and%20genomics%20for%20cancer%20prognosis%3A%20a%20brief%20review&journal=Chin.%20J.%20Cancer%20Res.&doi=10.21147%2Fj.issn.1000-9604.2021.05.03&volume=33&pages=563-573&publication_year=2021&author=Lu%2CC&author=Shiradkar%2CR&author=Liu%2CZ)
64. Guo, J. et al. Cross-modal deep learning model for predicting pathologic complete response to neoadjuvant chemotherapy in breast cancer. *NPJ Precis. Onc.* **8**, 189 (2024).

[Article](https://doi.org/10.1038%2Fs41698-024-00678-8)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Cross-modal%20deep%20learning%20model%20for%20predicting%20pathologic%20complete%20response%20to%20neoadjuvant%20chemotherapy%20in%20breast%20cancer&journal=NPJ%20Precis.%20Onc.&doi=10.1038%2Fs41698-024-00678-8&volume=8&publication_year=2024&author=Guo%2CJ)
65. Goyal, M. et al. A multi-model approach integrating whole-slide imaging and clinicopathologic features to predict breast cancer recurrence risk. *NPJ Breast Cancer* **10**, 93 (2024).

[Article](https://doi.org/10.1038%2Fs41523-024-00700-z)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXitlSitbjK)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=39426965)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11490577)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20multi-model%20approach%20integrating%20whole-slide%20imaging%20and%20clinicopathologic%20features%20to%20predict%20breast%20cancer%20recurrence%20risk&journal=NPJ%20Breast%20Cancer&doi=10.1038%2Fs41523-024-00700-z&volume=10&publication_year=2024&author=Goyal%2CM)
66. Joshi, G. et al. FDA-approved artificial intelligence and machine learning (AI/ML)-enabled medical devices: an updated landscape. *Electronics* **13**, 498 (2024).

[Article](https://doi.org/10.3390%2Felectronics13030498)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=FDA-approved%20artificial%20intelligence%20and%20machine%20learning%20%28AI%2FML%29-enabled%20medical%20devices%3A%20an%20updated%20landscape&journal=Electronics&doi=10.3390%2Felectronics13030498&volume=13&publication_year=2024&author=Joshi%2CG)
67. FDA. Artificial intelligence and machine learning (AI/ML)-enabled medical devices. *FDA* [https://www.fda.gov/medical-devices/software-medical-device-samd/artificial-intelligence-and-machine-learning-aiml-enabled-medical-devices](https://www.fda.gov/medical-devices/software-medical-device-samd/artificial-intelligence-and-machine-learning-aiml-enabled-medical-devices) (2024).
68. Cheng, J. Y., Abel, J. T., Balis, U. G. J., McClintock, D. S. & Pantanowitz, L. Challenges in the development, deployment, and regulation of artificial intelligence in anatomic pathology. *Am. J. Pathol.* **191**, 1684–1692 (2021).

[Article](https://doi.org/10.1016%2Fj.ajpath.2020.10.018)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33245914)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Challenges%20in%20the%20development%2C%20deployment%2C%20and%20regulation%20of%20artificial%20intelligence%20in%20anatomic%20pathology&journal=Am.%20J.%20Pathol.&doi=10.1016%2Fj.ajpath.2020.10.018&volume=191&pages=1684-1692&publication_year=2021&author=Cheng%2CJY&author=Abel%2CJT&author=Balis%2CUGJ&author=McClintock%2CDS&author=Pantanowitz%2CL)
69. European Commission. *Proposal for a Regulation of the European Parliament and of the Council Laying Down Harmonised Rules on Artificial Intelligence (Artificial Intelligence Act) and Amending Certain Union Legislative Acts* (Publications Office of the European Union, 2021).
70. Proscia. Proscia receives FDA 510(k) clearance for Concentriq AP-Dx. *Proscia* [proscia.com/press-releases/proscia-receives-fda-510k-clearance-for-concentriq-ap-dx-2/](https://proscia.com/press-releases/proscia-receives-fda-510k-clearance-for-concentriq-ap-dx-2/) (2024).
71. IBEX. AI-First Approach to Cancer Diagnostics Earns Ibex Medical Analytics Notable Milestones. *IBEX* [ibex-ai.com/milestones/](https://ibex-ai.com/milestones/) (2024).
72. Directorate-General for Health and Food Safety. Manufacturer IVD. *ec.europa.eu* [https://health.ec.europa.eu/medical-devices-new-regulations/getting-ready-new-regulations/manufacturer-ivd\_en](https://health.ec.europa.eu/medical-devices-new-regulations/getting-ready-new-regulations/manufacturer-ivd_en) (accessed 7 February 2025).
73. Food and Drug Administration. FDA Authorizes Software that Can Help Identify Prostate Cancer. *FDA* [www.fda.gov/news-events/press-announcements/fda-authorizes-software-can-help-identify-prostate-cancer](https://www.fda.gov/news-events/press-announcements/fda-authorizes-software-can-help-identify-prostate-cancer) (2021).
74. Roche Diagnostics. Roche Receives FDA Clearance on its Digital Pathology Solution for Diagnostic Use. *Roche Diagnostics* [diagnostics.roche.com/us/en/news-listing/2024/roche-receives-fda-clearance-on-its-digital-pathology-solution-for-diagnostic-use.html](https://diagnostics.roche.com/us/en/news-listing/2024/roche-receives-fda-clearance-on-its-digital-pathology-solution-for-diagnostic-use.html) (2024).
75. American Medical Association. CPT® Overview and Code Approval. *AMA* [www.ama-assn.org/practice-management/cpt/cpt-overview-and-code-approval](https://www.ama-assn.org/practice-management/cpt/cpt-overview-and-code-approval) (2024).
76. Centers for Medicare & Medicaid Services. Medicare Claims Processing Manual. *CMS.gov* [www.cms.gov/Regulations-and-Guidance/Guidance/Manuals/Internet-Only-Manuals-IOMs-Items/CMS018912](http://www.cms.gov/Regulations-and-Guidance/Guidance/Manuals/Internet-Only-Manuals-IOMs-Items/CMS018912) (2024).
77. College of American Pathologists. 13 Digital Pathology Codes you Should Use in 2023 and Beyond. *College of American Pathologists* [www.cap.org/member-resources/articles/13-digital-pathology-codes-you-should-use-in-2023-and-beyond](http://www.cap.org/member-resources/articles/13-digital-pathology-codes-you-should-use-in-2023-and-beyond) (2023).
78. College of American Pathologists. Digital Pathology Codes. *College of American Pathologists* [www.cap.org/advocacy/payments-for-pathology-services/digital-pathology-codes](https://www.cap.org/advocacy/payments-for-pathology-services/digital-pathology-codes) (2023).
79. College of American Pathologists. How to Use the New Digital Pathology CPT Codes: CAP Answers Questions. *College of American Pathologists* [www.cap.org/member-resources/articles/how-to-use-the-new-digital-pathology-cpt-codes](http://www.cap.org/member-resources/articles/how-to-use-the-new-digital-pathology-cpt-codes) (2023).
80. Mantovani, A., Leopaldi, C., Nighswander, C. M. & Bidino, R. D. Access and reimbursement pathways for digital health solutions and in vitro diagnostic devices: current scenario and challenges. *Front. Med. Technol.* **5**, 1101476 (2023).

[Article](https://doi.org/10.3389%2Ffmedt.2023.1101476)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36891483)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9986593)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Access%20and%20reimbursement%20pathways%20for%20digital%20health%20solutions%20and%20in%20vitro%20diagnostic%20devices%3A%20current%20scenario%20and%20challenges&journal=Front.%20Med.%20Technol.&doi=10.3389%2Ffmedt.2023.1101476&volume=5&publication_year=2023&author=Mantovani%2CA&author=Leopaldi%2CC&author=Nighswander%2CCM&author=Bidino%2CRD)
81. Pharmaceuticals and Medical Devices Agency. The 1st Subcommittee on Software as a Medical Device Utilizing AI and Machine Learning. *PMDA* [www.pmda.go.jp/english/rs-sb-std/sb/subcommittees/0024.html](http://www.pmda.go.jp/english/rs-sb-std/sb/subcommittees/0024.html) (2021).
82. Ministry of Food and Drug Safety. Guidance on the Review and Approval of Artificial Intelligence (AI)-based Medical Devices. *Ministry of Food and Drug Safety* [www.mfds.go.kr/eng/brd/m\_40/view.do?seq=72627&utm\_source=chatgpt.com](http://www.mfds.go.kr/eng/brd/m_40/view.do?seq=72627&utm_source=chatgpt.com) (2023).
83. National Medical Products Administration. NMPA Announcement on Guidance for the Classification Defining of AI-Based Medical Software Products. *NMPA* [english.nmpa.gov.cn/2021-07/08/c\_660267.htm?utm\_source=chatgpt.com](http://english.nmpa.gov.cn/2021-07/08/c_660267.htm?utm_source=chatgpt.com) (2021).
84. Indian Council of Medical Research. Ethical Guidelines for Application of Artificial Intelligence in Biomedical Research and Healthcare. *ICMR* [www.icmr.gov.in/ethical-guidelines-for-application-of-artificial-intelligence-in-biomedical-research-and-healthcare](http://www.icmr.gov.in/ethical-guidelines-for-application-of-artificial-intelligence-in-biomedical-research-and-healthcare) (2023).
85. Larsen, M. P. & Tsang, P. Laboratory Developed Tests. *PathologyOutlines.com* [www.pathologyoutlines.com/topic/managementlabdevelopedtests.html](http://www.pathologyoutlines.com/topic/managementlabdevelopedtests.html) (2024).
86. Food and Drug Administration. Laboratory Developed Tests. *FDA* [www.fda.gov/medical-devices/in-vitro-diagnostics/laboratory-developed-tests](https://www.fda.gov/medical-devices/in-vitro-diagnostics/laboratory-developed-tests) (2025).
87. Artera. For Patients. *Artera* [https://artera.ai/for-patients](https://artera.ai/for-patients) (2025).
88. Codify. CPT® Code 0376U, Under Proprietary Laboratory Analyses. *AAPC* [www.aapc.com/codes/cpt-codes/0376U](http://www.aapc.com/codes/cpt-codes/0376U) (2024).
89. Schaeffer, E. M. et al. NCCN guidelines® insights: prostate cancer, version 3.2024. *J. Natl Compr. Cancer Netw.* **22**, 140–150 (2024).

[Article](https://doi.org/10.6004%2Fjnccn.2024.0019)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=NCCN%20guidelines%C2%AE%20insights%3A%20prostate%20cancer%2C%20version%203.2024&journal=J.%20Natl%20Compr.%20Cancer%20Netw.&doi=10.6004%2Fjnccn.2024.0019&volume=22&pages=140-150&publication_year=2024&author=Schaeffer%2CEM)
90. Backman, I. The FDA’s Proposed Ruling on Lab Tests Could Have Unintended Consequences. *Yale School of Medicine* [medicine.yale.edu/news-article/the-fdas-proposed-ruling-on-lab-tests-could-have-unintended-consequences/](http://medicine.yale.edu/news-article/the-fdas-proposed-ruling-on-lab-tests-could-have-unintended-consequences/) (2024).
91. Dombrink, I. et al. Critical implications of IVDR for innovation in diagnostics: input from the Biomed Alliance Diagnostics Task Force. *HemaSphere* **6**, e724 (2022).

[Article](https://doi.org/10.1097%2FHS9.0000000000000724)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35620593)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9126521)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Critical%20implications%20of%20IVDR%20for%20innovation%20in%20diagnostics%3A%20input%20from%20the%20Biomed%20Alliance%20Diagnostics%20Task%20Force&journal=HemaSphere&doi=10.1097%2FHS9.0000000000000724&volume=6&publication_year=2022&author=Dombrink%2CI)
92. Yousif, M., Hassell, L. & Pantanowitz, L. in *Digital Innovation for Healthcare in COVID-19 Pandemic* (eds de Pablos, P. O. et al.) Ch. 7, 95–107 (Academic Press, 2022).
93. Luchini, C., Pea, A. & Scarpa, A. Artificial intelligence in oncology: current applications and future perspectives. *Br. J. Cancer* **126**, 4–9 (2022).

[Article](https://doi.org/10.1038%2Fs41416-021-01633-1)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34837074)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Artificial%20intelligence%20in%20oncology%3A%20current%20applications%20and%20future%20perspectives&journal=Br.%20J.%20Cancer&doi=10.1038%2Fs41416-021-01633-1&volume=126&pages=4-9&publication_year=2022&author=Luchini%2CC&author=Pea%2CA&author=Scarpa%2CA)
94. Wu, S. et al. An artificial intelligence model for detecting pathological lymph node metastasis in prostate cancer using whole slide images: a retrospective, multicentre, diagnostic study. *eClinicalMedicine* **71**, 102580 (2024).

[Article](https://doi.org/10.1016%2Fj.eclinm.2024.102580)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38618206)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11015342)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=An%20artificial%20intelligence%20model%20for%20detecting%20pathological%20lymph%20node%20metastasis%20in%20prostate%20cancer%20using%20whole%20slide%20images%3A%20a%20retrospective%2C%20multicentre%2C%20diagnostic%20study&journal=eClinicalMedicine&doi=10.1016%2Fj.eclinm.2024.102580&volume=71&publication_year=2024&author=Wu%2CS)
95. Reddy, S., Shaheed, A., Seo, Y. & Patel, R. Development of an artificial intelligence model for the classification of gastric carcinoma stages using pathology slides. *Cureus* **16**, e56740 (2024).

[PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38650818)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11033212)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Development%20of%20an%20artificial%20intelligence%20model%20for%20the%20classification%20of%20gastric%20carcinoma%20stages%20using%20pathology%20slides&journal=Cureus&volume=16&publication_year=2024&author=Reddy%2CS&author=Shaheed%2CA&author=Seo%2CY&author=Patel%2CR)
96. van Dooijeweert, C. et al. Clinical implementation of artificial-intelligence-assisted detection of breast cancer metastases in sentinel lymph nodes: the CONFIDENT-B single-center, non-randomized clinical trial. *Nat. Cancer* **5**, 1195–1205 (2024).

[Article](https://doi.org/10.1038%2Fs43018-024-00788-z)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=38937624)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11358151)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Clinical%20implementation%20of%20artificial-intelligence-assisted%20detection%20of%20breast%20cancer%20metastases%20in%20sentinel%20lymph%20nodes%3A%20the%20CONFIDENT-B%20single-center%2C%20non-randomized%20clinical%20trial&journal=Nat.%20Cancer&doi=10.1038%2Fs43018-024-00788-z&volume=5&pages=1195-1205&publication_year=2024&author=Dooijeweert%2CC)
97. Eloy, C. et al. Artificial intelligence-assisted cancer diagnosis improves the efficiency of pathologists in prostatic biopsies. *Virchows Arch.* **482**, 595–604 (2023).

[Article](https://link.springer.com/doi/10.1007/s00428-023-03518-5)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXjslentbg%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36809483)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10033575)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Artificial%20intelligence-assisted%20cancer%20diagnosis%20improves%20the%20efficiency%20of%20pathologists%20in%20prostatic%20biopsies&journal=Virchows%20Arch.&doi=10.1007%2Fs00428-023-03518-5&volume=482&pages=595-604&publication_year=2023&author=Eloy%2CC)
98. Dawson, H. Digital pathology – rising to the challenge. *Front. Med.* **9**, 888896 (2022).

[Article](https://doi.org/10.3389%2Ffmed.2022.888896)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Digital%20pathology%20%E2%80%93%20rising%20to%20the%20challenge&journal=Front.%20Med.&doi=10.3389%2Ffmed.2022.888896&volume=9&publication_year=2022&author=Dawson%2CH)
99. Williams, B. J., Lee, J., Oien, K. A. & Treanor, D. Digital pathology access and usage in the UK: results from a national survey on behalf of the National Cancer Research Institute’s CM-Path initiative. *J. Clin. Pathol.* **71**, 463–466 (2018).

[Article](https://doi.org/10.1136%2Fjclinpath-2017-204808)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29317516)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Digital%20pathology%20access%20and%20usage%20in%20the%20UK%3A%20results%20from%20a%20national%20survey%20on%20behalf%20of%20the%20National%20Cancer%20Research%20Institute%E2%80%99s%20CM-Path%20initiative&journal=J.%20Clin.%20Pathol.&doi=10.1136%2Fjclinpath-2017-204808&volume=71&pages=463-466&publication_year=2018&author=Williams%2CBJ&author=Lee%2CJ&author=Oien%2CKA&author=Treanor%2CD)
100. Aggarwal, A., Badve, S. S., Cuzick, J., Madabhushi, A. & Thorat, M. A. A computational pathology collagen signature predictive of tamoxifen benefit in ductal carcinoma in situ: results from a cohort within the UK/ANZ DCIS randomized trial \[abstract 171P\]. *Ann. Oncol.* **35** (Suppl. 2), S284 (2024).

[Article](https://doi.org/10.1016%2Fj.annonc.2024.08.2176)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20computational%20pathology%20collagen%20signature%20predictive%20of%20tamoxifen%20benefit%20in%20ductal%20carcinoma%20in%20situ%3A%20results%20from%20a%20cohort%20within%20the%20UK%2FANZ%20DCIS%20randomized%20trial%20%5Babstract%20171P%5D&journal=Ann.%20Oncol.&doi=10.1016%2Fj.annonc.2024.08.2176&volume=35&issue=Suppl.%202&publication_year=2024&author=Aggarwal%2CA&author=Badve%2CSS&author=Cuzick%2CJ&author=Madabhushi%2CA&author=Thorat%2CMA)
101. Spratt, D. E. et al. Patient-level data meta-analysis of a multi-modal artificial intelligence (MMAI) prognostic biomarker in high-risk prostate cancer: results from six NRG/RTOG phase III randomized trials \[abstract\]. *J. Clin. Oncol.* **41** (Suppl. 6), 299 (2023).

[Article](https://doi.org/10.1200%2FJCO.2023.41.6_suppl.299)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Patient-level%20data%20meta-analysis%20of%20a%20multi-modal%20artificial%20intelligence%20%28MMAI%29%20prognostic%20biomarker%20in%20high-risk%20prostate%20cancer%3A%20results%20from%20six%20NRG%2FRTOG%20phase%20III%20randomized%20trials%20%5Babstract%5D&journal=J.%20Clin.%20Oncol.&doi=10.1200%2FJCO.2023.41.6_suppl.299&volume=41&issue=Suppl.%206&publication_year=2023&author=Spratt%2CDE)
102. Vora, L. K. et al. Artificial intelligence in pharmaceutical technology and drug delivery design. *Pharmaceutics* **15**, 1916 (2023).

[Article](https://doi.org/10.3390%2Fpharmaceutics15071916)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhs1KnsbrI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37514102)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10385763)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Artificial%20intelligence%20in%20pharmaceutical%20technology%20and%20drug%20delivery%20design&journal=Pharmaceutics&doi=10.3390%2Fpharmaceutics15071916&volume=15&publication_year=2023&author=Vora%2CLK)
103. Lam, T. Y. T. et al. Randomized controlled trials of artificial intelligence in clinical practice: systematic review. *J. Med. Internet Res.* **24**, e37188 (2022).

[Article](https://doi.org/10.2196%2F37188)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35904087)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9459941)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Randomized%20controlled%20trials%20of%20artificial%20intelligence%20in%20clinical%20practice%3A%20systematic%20review&journal=J.%20Med.%20Internet%20Res.&doi=10.2196%2F37188&volume=24&publication_year=2022&author=Lam%2CTYT)
104. Kiran, N. et al. Digital pathology: transforming diagnosis in the digital age. *Cureus* **15**, e44620 (2023).

[PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37799211)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10547926)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Digital%20pathology%3A%20transforming%20diagnosis%20in%20the%20digital%20age&journal=Cureus&volume=15&publication_year=2023&author=Kiran%2CN)
105. Zarella, M. D. et al. A practical guide to whole slide imaging: a white paper from the digital pathology association. *Arch. Pathol. Laboratory Med.* **143**, 222–234 (2019).

[Article](https://doi.org/10.5858%2Farpa.2018-0343-RA)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20practical%20guide%20to%20whole%20slide%20imaging%3A%20a%20white%20paper%20from%20the%20digital%20pathology%20association&journal=Arch.%20Pathol.%20Laboratory%20Med.&doi=10.5858%2Farpa.2018-0343-RA&volume=143&pages=222-234&publication_year=2019&author=Zarella%2CMD)
106. Pantanowitz, L. Digital images and the future of digital pathology. *J. Pathol. Inform.* **1**, 15 (2010).

[Article](https://doi.org/10.4103%2F2153-3539.68332)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20922032)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2941968)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Digital%20images%20and%20the%20future%20of%20digital%20pathology&journal=J.%20Pathol.%20Inform.&doi=10.4103%2F2153-3539.68332&volume=1&publication_year=2010&author=Pantanowitz%2CL)
107. Morales, S., Engan, K. & Naranjo, V. Artificial intelligence in computational pathology – challenges and future directions. *Digital Signal. Process.* **119**, 103196 (2021).

[Article](https://doi.org/10.1016%2Fj.dsp.2021.103196)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Artificial%20intelligence%20in%20computational%20pathology%20%E2%80%93%20challenges%20and%20future%20directions&journal=Digital%20Signal.%20Process.&doi=10.1016%2Fj.dsp.2021.103196&volume=119&publication_year=2021&author=Morales%2CS&author=Engan%2CK&author=Naranjo%2CV)
108. Fitt, I. Digital Pathology – World – 2023. *Signify Research* [www.signifyresearch.net/market-intelligence/digital-pathology-world-2023/](https://www.signifyresearch.net/market-intelligence/digital-pathology-world-2023/) (2023).
109. Law, R. Roche gains 510(k) clearance for digital pathology solution for diagnostic use. *Medical Device Network* [https://www.medicaldevice-network.com/news/roche-gains-501-clearance-for-digital-pathology-solution-for-diagnostic-use/?cf-view](https://www.medicaldevice-network.com/news/roche-gains-501-clearance-for-digital-pathology-solution-for-diagnostic-use/?cf-view) (2024).
110. Abbvie. Precision Medicine. *Abbvie* [www.abbvie.com/science/areas-of-innovation/precision-medicine.html](http://www.abbvie.com/science/areas-of-innovation/precision-medicine.html) (2024).
111. Berbís, M. Á. Digital and computational pathology: what a time to be alive! *Mayo Clin. Proc. Digital Health* **2**, 18–20 (2024).

[Article](https://doi.org/10.1016%2Fj.mcpdig.2023.11.006)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Digital%20and%20computational%20pathology%3A%20what%20a%20time%20to%20be%20alive%21&journal=Mayo%20Clin.%20Proc.%20Digital%20Health&doi=10.1016%2Fj.mcpdig.2023.11.006&volume=2&pages=18-20&publication_year=2024&author=Berb%C3%ADs%2CM%C3%81)
112. García-Rivello, H. et al. Digital pathology in Latin America. *Surg. Exp. Pathol.* **6**, 10 (2023).

[Article](https://link.springer.com/doi/10.1186/s42047-023-00135-z)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Digital%20pathology%20in%20Latin%20America&journal=Surg.%20Exp.%20Pathol.&doi=10.1186%2Fs42047-023-00135-z&volume=6&publication_year=2023&author=Garc%C3%ADa-Rivello%2CH)
113. Baidoshvili, A. et al. Evaluating the benefits of digital pathology implementation: time savings in laboratory logistics. *Histopathology* **73**, 784–794 (2018).

[Article](https://doi.org/10.1111%2Fhis.13691)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29924891)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Evaluating%20the%20benefits%20of%20digital%20pathology%20implementation%3A%20time%20savings%20in%20laboratory%20logistics&journal=Histopathology&doi=10.1111%2Fhis.13691&volume=73&pages=784-794&publication_year=2018&author=Baidoshvili%2CA)
114. Hanna, M. G. et al. Implementation of digital pathology offers clinical and operational increase in efficiency and cost savings. *Arch. Pathol. Lab. Med.* **143**, 1545–1555 (2019).

[Article](https://doi.org/10.5858%2Farpa.2018-0514-OA)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31173528)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7448534)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Implementation%20of%20digital%20pathology%20offers%20clinical%20and%20operational%20increase%20in%20efficiency%20and%20cost%20savings&journal=Arch.%20Pathol.%20Lab.%20Med.&doi=10.5858%2Farpa.2018-0514-OA&volume=143&pages=1545-1555&publication_year=2019&author=Hanna%2CMG)
115. Vergani, A., Regis, B., Jocollé, G., Patetta, R. & Rossi, G. Noninferiority diagnostic value, but also economic and turnaround time advantages from digital pathology. *Am. J. Surg. Pathol.* **42**, 841 (2018).

[Article](https://doi.org/10.1097%2FPAS.0000000000001035)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29438166)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Noninferiority%20diagnostic%20value%2C%20but%20also%20economic%20and%20turnaround%20time%20advantages%20from%20digital%20pathology&journal=Am.%20J.%20Surg.%20Pathol.&doi=10.1097%2FPAS.0000000000001035&volume=42&publication_year=2018&author=Vergani%2CA&author=Regis%2CB&author=Jocoll%C3%A9%2CG&author=Patetta%2CR&author=Rossi%2CG)
116. Zarella, M. D. et al. Artificial intelligence and digital pathology: clinical promise and deployment considerations. *J. Med. Imaging* **10**, 051802 (2023).

[Article](https://doi.org/10.1117%2F1.JMI.10.5.051802)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Artificial%20intelligence%20and%20digital%20pathology%3A%20clinical%20promise%20and%20deployment%20considerations&journal=J.%20Med.%20Imaging&doi=10.1117%2F1.JMI.10.5.051802&volume=10&publication_year=2023&author=Zarella%2CMD)
117. Pinto, D. G., Bychkov, A., Tsuyama, N., Fukuoka, J. & Eloy, C. Real-world implementation of digital pathology: results from an intercontinental survey. *Lab. Invest.* **103**, 100261 (2023).

[Article](https://doi.org/10.1016%2Fj.labinv.2023.100261)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37839634)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Real-world%20implementation%20of%20digital%20pathology%3A%20results%20from%20an%20intercontinental%20survey&journal=Lab.%20Invest.&doi=10.1016%2Fj.labinv.2023.100261&volume=103&publication_year=2023&author=Pinto%2CDG&author=Bychkov%2CA&author=Tsuyama%2CN&author=Fukuoka%2CJ&author=Eloy%2CC)
118. Scalco, R. et al. The status of digital pathology and associated infrastructure within Alzheimer’s disease centers. *J. Neuropathol. Exp. Neurol.* **82**, 202–211 (2023).

[Article](https://doi.org/10.1093%2Fjnen%2Fnlac127)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36692179)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9941826)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20status%20of%20digital%20pathology%20and%20associated%20infrastructure%20within%20Alzheimer%E2%80%99s%20disease%20centers&journal=J.%20Neuropathol.%20Exp.%20Neurol.&doi=10.1093%2Fjnen%2Fnlac127&volume=82&pages=202-211&publication_year=2023&author=Scalco%2CR)
119. Indu, M., Rathy, R. & Binu, M. P. “Slide less pathology”: fairy tale or reality? *J. Oral. Maxillofac. Pathol.* **20**, 284 (2016).

[Article](https://doi.org/10.4103%2F0973-029X.185921)  [CAS](https://www.nature.com/articles/cas-redirect/1:STN:280:DC%2BC2szotlGmtQ%3D%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27601824)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4989562)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=%E2%80%9CSlide%20less%20pathology%E2%80%9D%3A%20fairy%20tale%20or%20reality%3F&journal=J.%20Oral.%20Maxillofac.%20Pathol.&doi=10.4103%2F0973-029X.185921&volume=20&publication_year=2016&author=Indu%2CM&author=Rathy%2CR&author=Binu%2CMP)
120. Bhargava, R. & Madabhushi, A. Emerging themes in image informatics and molecular analysis for digital pathology. *Annu. Rev. Biomed. Eng.* **18**, 387–412 (2016).

[Article](https://doi.org/10.1146%2Fannurev-bioeng-112415-114722)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhtFyhsrnO)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27420575)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5533658)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Emerging%20themes%20in%20image%20informatics%20and%20molecular%20analysis%20for%20digital%20pathology&journal=Annu.%20Rev.%20Biomed.%20Eng.&doi=10.1146%2Fannurev-bioeng-112415-114722&volume=18&pages=387-412&publication_year=2016&author=Bhargava%2CR&author=Madabhushi%2CA)
121. Liu, Y., Levenson, R. M. & Jenkins, M. W. Slide over: advances in slide-free optical microscopy as drivers of diagnostic pathology. *Am. J. Pathol.* **192**, 180–194 (2022).

[Article](https://doi.org/10.1016%2Fj.ajpath.2021.10.010)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34774514)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8883436)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Slide%20over%3A%20advances%20in%20slide-free%20optical%20microscopy%20as%20drivers%20of%20diagnostic%20pathology&journal=Am.%20J.%20Pathol.&doi=10.1016%2Fj.ajpath.2021.10.010&volume=192&pages=180-194&publication_year=2022&author=Liu%2CY&author=Levenson%2CRM&author=Jenkins%2CMW)
122. Indica Labs. Indica Labs Receives First FDA Clearance for HALO AP Dx Digital Pathology Platform. *Indica Labs* [indicalab.com/news/indica-labs-receives-first-fda-clearance-for-halo-ap-dx-digital-pathology-platform/](https://indicalab.com/news/indica-labs-receives-first-fda-clearance-for-halo-ap-dx-digital-pathology-platform/) (2024).
123. Hosseini, M. S. et al. Computational pathology: a survey review and the way forward. *J. Pathol. Inf.* **15**, 100357 (2024).

[Article](https://doi.org/10.1016%2Fj.jpi.2023.100357)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Computational%20pathology%3A%20a%20survey%20review%20and%20the%20way%20forward&journal=J.%20Pathol.%20Inf.&doi=10.1016%2Fj.jpi.2023.100357&volume=15&publication_year=2024&author=Hosseini%2CMS)
124. Ghassemi, M., Oakden-Rayner, L. & Beam, A. L. The false hope of current approaches to explainable artificial intelligence in health care. *Lancet Digital Health* **3**, e745–e750 (2021).

[Article](https://doi.org/10.1016%2FS2589-7500%2821%2900208-9)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXis1Sltbk%3D)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34711379)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20false%20hope%20of%20current%20approaches%20to%20explainable%20artificial%20intelligence%20in%20health%20care&journal=Lancet%20Digital%20Health&doi=10.1016%2FS2589-7500%2821%2900208-9&volume=3&pages=e745-e750&publication_year=2021&author=Ghassemi%2CM&author=Oakden-Rayner%2CL&author=Beam%2CAL)
125. Chauhan, C. & Gullapalli, R. R. Ethics of AI in pathology: current paradigms and emerging issues. *Am. J. Pathol.* **191**, 1673–1683 (2021).

[Article](https://doi.org/10.1016%2Fj.ajpath.2021.06.011)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34252382)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8485059)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Ethics%20of%20AI%20in%20pathology%3A%20current%20paradigms%20and%20emerging%20issues&journal=Am.%20J.%20Pathol.&doi=10.1016%2Fj.ajpath.2021.06.011&volume=191&pages=1673-1683&publication_year=2021&author=Chauhan%2CC&author=Gullapalli%2CRR)
126. Roberts, H. et al. in *Ethics, Governance, and Policies in Artificial Intelligence* (ed. Floridi, L.) 47–79 (Springer, 2021).
127. Habuka, H. *Japan’s Approach to AI Regulation and its Impact on the 2023 G7 Presidency* (Center for Strategic & International Studies, 2023).
128. Chouffani El Fassi, S. et al. Not all AI health tools with regulatory authorization are clinically validated. *Nat. Med.* **30**, 2718–2720 (2024).

[Article](https://doi.org/10.1038%2Fs41591-024-03203-3)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXhvVekur%2FN)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=39187696)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Not%20all%20AI%20health%20tools%20with%20regulatory%20authorization%20are%20clinically%20validated&journal=Nat.%20Med.&doi=10.1038%2Fs41591-024-03203-3&volume=30&pages=2718-2720&publication_year=2024&author=Chouffani%20El%20Fassi%2CS)
129. Businesswire. ArteraAI Announced as the First-and-Only Predictive Test for Therapy Personalization in the 2024 NCCN Guidelines® for Prostate Cancer. *Businesswire* [www.businesswire.com/news/home/20240304893588/en/ArteraAI-Announced-as-the-First-and-Only-Predictive-Test-for-Therapy-Personalization-in-the-2024-NCCN-Guidelines%C2%AE-for-Prostate-Cancer](https://www.businesswire.com/news/home/20240304893588/en/ArteraAI-Announced-as-the-First-and-Only-Predictive-Test-for-Therapy-Personalization-in-the-2024-NCCN-Guidelines%C2%AE-for-Prostate-Cancer) (2024).
130. Businesswire. Three UK-Based Hospital Systems Exploring the Power of Paige’s AI for Diagnosing Prostate Cancer in Live Clinical Settings. *Businesswire* [www.businesswire.com/news/home/20240722858117/en/Three-UK-Based-Hospital-Systems-Exploring-the-Power-of-Paige%E2%80%99s-AI-for-Diagnosing-Prostate-Cancer-in-Live-Clinical-Settings](https://www.businesswire.com/news/home/20240722858117/en/Three-UK-Based-Hospital-Systems-Exploring-the-Power-of-Paige%E2%80%99s-AI-for-Diagnosing-Prostate-Cancer-in-Live-Clinical-Settings) (2024).
131. Kim, I., Kang, K., Song, Y. & Kim, T.-J. Application of artificial intelligence in pathology: trends and challenges. *Diagnostics* **12**, 2794 (2022).

[Article](https://doi.org/10.3390%2Fdiagnostics12112794)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36428854)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9688959)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Application%20of%20artificial%20intelligence%20in%20pathology%3A%20trends%20and%20challenges&journal=Diagnostics&doi=10.3390%2Fdiagnostics12112794&volume=12&publication_year=2022&author=Kim%2CI&author=Kang%2CK&author=Song%2CY&author=Kim%2CT-J)
132. Kanan, C. et al. Independent validation of Paige Prostate: assessing clinical benefit of an artificial intelligence tool within a digital diagnostic pathology laboratory workflow \[abstract\]. *J. Clin. Oncol.* **38** (Suppl. 15), e14076 (2020).

[Article](https://doi.org/10.1200%2FJCO.2020.38.15_suppl.e14076)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Independent%20validation%20of%20Paige%20Prostate%3A%20assessing%20clinical%20benefit%20of%20an%20artificial%20intelligence%20tool%20within%20a%20digital%20diagnostic%20pathology%20laboratory%20workflow%20%5Babstract%5D&journal=J.%20Clin.%20Oncol.&doi=10.1200%2FJCO.2020.38.15_suppl.e14076&volume=38&issue=Suppl.%2015&publication_year=2020&author=Kanan%2CC)
133. Zehra, T., Parwani, A., Abdul-Ghafar, J. & Ahmad, Z. A suggested way forward for adoption of AI-enabled digital pathology in low resource organizations in the developing world. *Diagn. Pathol.* **18**, 68 (2023).

[Article](https://link.springer.com/doi/10.1186/s13000-023-01352-6)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37202805)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10193335)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20suggested%20way%20forward%20for%20adoption%20of%20AI-enabled%20digital%20pathology%20in%20low%20resource%20organizations%20in%20the%20developing%20world&journal=Diagn.%20Pathol.&doi=10.1186%2Fs13000-023-01352-6&volume=18&publication_year=2023&author=Zehra%2CT&author=Parwani%2CA&author=Abdul-Ghafar%2CJ&author=Ahmad%2CZ)
134. Schwen, L. O., Kiehl, T.-R., Carvalho, R., Zerbe, N. & Homeyer, A. Digitization of pathology labs: a review of lessons learned. *Lab. Invest.* **103**, 100244 (2023).

[Article](https://doi.org/10.1016%2Fj.labinv.2023.100244)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=37657651)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Digitization%20of%20pathology%20labs%3A%20a%20review%20of%20lessons%20learned&journal=Lab.%20Invest.&doi=10.1016%2Fj.labinv.2023.100244&volume=103&publication_year=2023&author=Schwen%2CLO&author=Kiehl%2CT-R&author=Carvalho%2CR&author=Zerbe%2CN&author=Homeyer%2CA)
135. Zhang, D. Y. et al. Implementation of digital pathology and artificial intelligence in routine pathology practice. *Lab. Invest.* **104**, 102111 (2024).

[Article](https://doi.org/10.1016%2Fj.labinv.2024.102111)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=39053633)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Implementation%20of%20digital%20pathology%20and%20artificial%20intelligence%20in%20routine%20pathology%20practice&journal=Lab.%20Invest.&doi=10.1016%2Fj.labinv.2024.102111&volume=104&publication_year=2024&author=Zhang%2CDY)
136. Sparano, J. A. et al. Adjuvant chemotherapy guided by a 21-gene expression assay in breast cancer. *N. Engl. J. Med.* **379**, 111–121 (2018).

[Article](https://doi.org/10.1056%2FNEJMoa1804710)  [CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtlert7rI)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29860917)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6172658)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Adjuvant%20chemotherapy%20guided%20by%20a%2021-gene%20expression%20assay%20in%20breast%20cancer&journal=N.%20Engl.%20J.%20Med.&doi=10.1056%2FNEJMoa1804710&volume=379&pages=111-121&publication_year=2018&author=Sparano%2CJA)
137. Food and Drug Administration. 510(k) Premarket Notification. *FDA* [www.accessdata.fda.gov/scripts/cdrh/cfdocs/cfpmn/pmn.cfm?ID=K212361](https://www.accessdata.fda.gov/scripts/cdrh/cfdocs/cfpmn/pmn.cfm?ID=K212361) (2024).
138. Food and Drug Administration. Premarket Approval (PMA). *FDA* [www.fda.gov/medical-devices/premarket-submissions-selecting-and-preparing-correct-submission/premarket-approval-pma](https://www.fda.gov/medical-devices/premarket-submissions-selecting-and-preparing-correct-submission/premarket-approval-pma) (2023).
139. Food and Drug Administration. De Novo Classification Request. *FDA* [www.fda.gov/medical-devices/premarket-submissions-selecting-and-preparing-correct-submission/de-novo-classification-request](http://www.fda.gov/medical-devices/premarket-submissions-selecting-and-preparing-correct-submission/de-novo-classification-request) (2022).
140. Directorate-General for Health and Food Safety. In-House Medical Devices. *European Commission* [health.ec.europa.eu/medical-devices-topics-interest/house-medical-devices\_en](https://health.ec.europa.eu/medical-devices-topics-interest/house-medical-devices_en) (2024).
141. Shurrab, S. & Duwairi, R. Self-supervised learning methods and applications in medical imaging analysis: a survey. *PeerJ Comput. Sci.* **8**, e1045 (2022).

[Article](https://doi.org/10.7717%2Fpeerj-cs.1045)  [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36091989)  [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9455147)  [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Self-supervised%20learning%20methods%20and%20applications%20in%20medical%20imaging%20analysis%3A%20a%20survey&journal=PeerJ%20Comput.%20Sci.&doi=10.7717%2Fpeerj-cs.1045&volume=8&publication_year=2022&author=Shurrab%2CS&author=Duwairi%2CR)

[Download references](https://citation-needed.springer.com/v2/references/10.1038/s41571-025-00991-6?format=refman&flavour=references)

## Acknowledgements

The research of A.M. is supported by the Lung Cancer Research Program (W81XWH-18-1-0440, W81XWH-20-1-0595), Mayo Clinic Breast Cancer SPORE grant P50 CA116201 from the National Institutes of Health, the Kidney Precision Medicine Project (KPMP) Glue Grant, National Cancer Institute (under award numbers R01CA249992-01A1, R01CA202752-01A1, R01CA208236-01A1, R01CA216579-01A1, R01CA220581-01A1, R01CA257612-01A1, 1U01CA239055-01, 1U01CA248226-01 and 1U54CA254566-01), National Center for Research Resources (under award number 1 C06 RR12463-01), National Heart, Lung and Blood Institute (1R01HL15127701A1 and R01HL15807101A1), National Institute of Biomedical Imaging and Bioengineering (1R43EB028736-01), Peer Reviewed Cancer Research Program (W81XWH-18-1-0404, W81XWH-21-1-0345 and W81XWH-21-1-0160), Prostate Cancer Research Program (W81XWH-15-1-0558 and W81XWH-20-1-0851). and the VA Merit Review Award IBX004121A from the United States Department of Veterans Affairs Biomedical Laboratory Research and Development Service the Office of the Assistant Secretary of Defense for Health Affairs, through the Breast Cancer Research Program (W81XWH-19-1-0668). The content is solely the responsibility of the authors and does not necessarily represent the official views of the National Institutes of Health, the US Department of Veterans Affairs, the Department of Defense, or the US Government.

## Author information

Author notes

1. These authors contributed equally: Arpit Aggarwal, Satvika Bharadwaj.

### Authors and Affiliations

1. Wallace H. Coulter Department of Biomedical Engineering, Emory University and Georgia Institute of Technology, Atlanta, GA, USA

Arpit Aggarwal, Satvika Bharadwaj, Germán Corredor & Anant Madabhushi
2. Atlanta Veterans Affairs Medical Center, Atlanta, GA, USA

Germán Corredor & Anant Madabhushi
3. Department of Biomedical Engineering, Emory University, Atlanta, GA, USA

Tilak Pathak
4. Department of Pathology and Laboratory Medicine, Emory University School of Medicine, Atlanta, GA, USA

Sunil Badve

Authors

1. Arpit Aggarwal

You can also search for this author in [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Arpit%20Aggarwal) [Google Scholar](http://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Arpit%20Aggarwal%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
2. Satvika Bharadwaj

You can also search for this author in [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Satvika%20Bharadwaj) [Google Scholar](http://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Satvika%20Bharadwaj%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
3. Germán Corredor

You can also search for this author in [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Germ%C3%A1n%20Corredor) [Google Scholar](http://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Germ%C3%A1n%20Corredor%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
4. Tilak Pathak

You can also search for this author in [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Tilak%20Pathak) [Google Scholar](http://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Tilak%20Pathak%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
5. Sunil Badve

You can also search for this author in [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Sunil%20Badve) [Google Scholar](http://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Sunil%20Badve%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
6. Anant Madabhushi

You can also search for this author in [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Anant%20Madabhushi) [Google Scholar](http://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Anant%20Madabhushi%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)

### Contributions

A.A. and S. Bharadwaj researched data for the article and wrote it. All authors contributed substantially to discussion of the contents, and reviewed and edited the manuscript before submission.

### Corresponding author

Correspondence to [Anant Madabhushi](https://www.nature.com/articles/).

## Ethics declarations

### Competing interests

S. Badve is on the advisory board for Mindpeak; and is also an ad hoc adviser for Agilent, AstraZeneca, Daichii-Sanyo and Roche-Ventana. A.M. is an equity holder in Picture Health, Elucid Bioimaging and Inspirata Inc.; currently he serves on the advisory board of Picture Health; has sponsored research agreements with AstraZeneca, Boehringer-Ingelheim, Bristol Myers-Squibb and Eli-Lilly; has developed technology licensed to Picture Health and Elucid Bioimaging; is involved in two different R01 grants with Inspirata Inc.; and is a member for the Frederick National Laboratory Advisory Committee. A.A., S. Bharadwaj, G.C. and T.P. declare no competing interests.

## Peer review

### Peer review information

*Nature Reviews Clinical Oncology* thanks J. N. Kather and the other, anonymous, reviewer(s) for their contribution to the peer review of this work.

## Additional information

**Publisher’s note** Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

## Glossary

510(k) clearance

Premarket approval submitted to the FDA to demonstrate that the device to be marketed is safe and effective<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 137" title="Food and Drug Administration. 510(k) Premarket Notification. FDA 
                www.accessdata.fda.gov/scripts/cdrh/cfdocs/cfpmn/pmn.cfm?ID=K212361
                
               (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR137" id="ref-link-section-d5807050e1357">137</a></sup>. Premarket approvals are conducted by the FDA to ensure the safety and effectiveness of Class III medical devices<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 138" title="Food and Drug Administration. Premarket Approval (PMA). FDA 
                www.fda.gov/medical-devices/premarket-submissions-selecting-and-preparing-correct-submission/premarket-approval-pma
                
               (2023)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR138" id="ref-link-section-d5807050e1361">138</a></sup>.

Convolutional neural networks

(CNNs). A type of deep learning model designed to process data, such as images<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Bera, K., Schalper, K. A., Rimm, D. L., Velcheti, V. &amp; Madabhushi, A. Artificial intelligence in digital pathology – new tools for diagnosis and precision oncology. Nat. Rev. Clin. Oncol. 16, 703–715 (2019)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR7" id="ref-link-section-d5807050e1376">7</a></sup>. CNNs use convolutional layers to automatically extract spatial features, such as edges, shapes and textures, making them highly effective for image-based tasks, including tumour segmentation, classification and pattern recognition in computational pathology<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Bera, K., Schalper, K. A., Rimm, D. L., Velcheti, V. &amp; Madabhushi, A. Artificial intelligence in digital pathology – new tools for diagnosis and precision oncology. Nat. Rev. Clin. Oncol. 16, 703–715 (2019)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR7" id="ref-link-section-d5807050e1380">7</a></sup>.

De Novo Classification Request

FDA pathway for the classification and marketing of Class I and II medical devices, which precede future 510(k) submissions<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 139" title="Food and Drug Administration. De Novo Classification Request. FDA 
                www.fda.gov/medical-devices/premarket-submissions-selecting-and-preparing-correct-submission/de-novo-classification-request
                
               (2022)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR139" id="ref-link-section-d5807050e1395">139</a></sup>.

Foundation models

Large-scale machine learning models trained on vast, unlabelled datasets that can be fine-tuned for various downstream tasks. In computational pathology, these models can be adapted for image classification, segmentation and prognostic prediction across different types of tissue, leveraging their broad generalization capabilities to enhance accuracy and adaptability in various clinical applications<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Chen, R. J. et al. Towards a general-purpose foundation model for computational pathology. Nat. Med. 30, 850–862 (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR11" id="ref-link-section-d5807050e1410">11</a></sup>.

In-house devices

In vitro diagnostic devices manufactured and used exclusively within health-care institutions in the European Union, exempt from Conformité Européene marking if they comply with the provisions of Article 5(5) of the In Vitro Diagnostic Medical Devices Regulation<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 140" title="Directorate-General for Health and Food Safety. In-House Medical Devices. European Commission 
                health.ec.europa.eu/medical-devices-topics-interest/house-medical-devices_en
                
               (2024)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR140" id="ref-link-section-d5807050e1426">140</a></sup>.

Laboratory-developed tests

In vitro diagnostic products intended for clinical use in the USA and designed, manufactured and used within a single laboratory certified under the Clinical Laboratory Improvement Amendments of 1988 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 86" title="Food and Drug Administration. Laboratory Developed Tests. FDA 
                www.fda.gov/medical-devices/in-vitro-diagnostics/laboratory-developed-tests
                
               (2025)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR86" id="ref-link-section-d5807050e1441">86</a></sup>).

Machine learning

An approach that enables the identification of patterns from data and the making of decisions or predictions without explicit programming. In computational pathology, machine learning approaches are used to analyse large-scale medical datasets, such as digitized slides, aiding in patient diagnosis, prognosis and treatment planning<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Bera, K., Schalper, K. A., Rimm, D. L., Velcheti, V. &amp; Madabhushi, A. Artificial intelligence in digital pathology – new tools for diagnosis and precision oncology. Nat. Rev. Clin. Oncol. 16, 703–715 (2019)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR7" id="ref-link-section-d5807050e1456">7</a></sup>.

Self-supervised learning models

A machine learning approach in which models learn from data without the need for manually labelled examples. These models are particularly valuable in computational pathology, an area in which annotated datasets are limited but large-scale image repositories are available<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 141" title="Shurrab, S. &amp; Duwairi, R. Self-supervised learning methods and applications in medical imaging analysis: a survey. PeerJ Comput. Sci. 8, e1045 (2022)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR141" id="ref-link-section-d5807050e1471">141</a></sup>.

Supervised learning models

A machine learning approach in which models learn from labelled data (that is, from datasets with known input–output pairs). This technique is often used in tasks for which labelled pathology images, such as tumour versus non-tumour regions, are available, enabling the model to classify new data on the basis of learned patterns<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Bera, K., Schalper, K. A., Rimm, D. L., Velcheti, V. &amp; Madabhushi, A. Artificial intelligence in digital pathology – new tools for diagnosis and precision oncology. Nat. Rev. Clin. Oncol. 16, 703–715 (2019)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR7" id="ref-link-section-d5807050e1486">7</a></sup>.

Transformers

Deep learning architectures designed for natural language processing tasks but increasingly applied to image analysis tasks<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 138" title="Food and Drug Administration. Premarket Approval (PMA). FDA 
                www.fda.gov/medical-devices/premarket-submissions-selecting-and-preparing-correct-submission/premarket-approval-pma
                
               (2023)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR138" id="ref-link-section-d5807050e1501">138</a></sup>. Transformers rely on a self-attention mechanism that enables them to capture relationships across all parts of the input image<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 138" title="Food and Drug Administration. Premarket Approval (PMA). FDA 
                www.fda.gov/medical-devices/premarket-submissions-selecting-and-preparing-correct-submission/premarket-approval-pma
                
               (2023)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR138" id="ref-link-section-d5807050e1505">138</a></sup>. In computational pathology, transformers are being used for tasks including whole-slide image classification<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 138" title="Food and Drug Administration. Premarket Approval (PMA). FDA 
                www.fda.gov/medical-devices/premarket-submissions-selecting-and-preparing-correct-submission/premarket-approval-pma
                
               (2023)." href="https://www.nature.com/articles/s41571-025-00991-6#ref-CR138" id="ref-link-section-d5807050e1509">138</a></sup>.

## Rights and permissions

Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law.

[Reprints and permissions](https://s100.copyright.com/AppDispatchServlet?title=Artificial%20intelligence%20in%20digital%20pathology%20%E2%80%94%20time%20for%20a%20reality%20check&author=Arpit%20Aggarwal%20et%20al&contentID=10.1038%2Fs41571-025-00991-6&copyright=Springer%20Nature%20Limited&publication=1759-4774&publicationDate=2025-02-11&publisherName=SpringerNature&orderBeanReset=true)

## About this article

[![Check for updates. Verify currency and authenticity via CrossMark](https://www.nature.com/articles/)](https://crossmark.crossref.org/dialog/?doi=10.1038/s41571-025-00991-6)

### Cite this article

Aggarwal, A., Bharadwaj, S., Corredor, G. *et al.* Artificial intelligence in digital pathology — time for a reality check. *Nat Rev Clin Oncol* (2025). https://doi.org/10.1038/s41571-025-00991-6

[Download citation](https://citation-needed.springer.com/v2/references/10.1038/s41571-025-00991-6?format=refman&flavour=citation)

- Accepted: 24 January 2025
- Published: 11 February 2025
- DOI: https://doi.org/10.1038/s41571-025-00991-6

### Subjects